{
  "data": [
    {
      "url": "https://api.github.com/repos/langgenius/dify/issues/20960",
      "repository_url": "https://api.github.com/repos/langgenius/dify",
      "labels_url": "https://api.github.com/repos/langgenius/dify/issues/20960/labels{/name}",
      "comments_url": "https://api.github.com/repos/langgenius/dify/issues/20960/comments",
      "events_url": "https://api.github.com/repos/langgenius/dify/issues/20960/events",
      "html_url": "https://github.com/langgenius/dify/issues/20960",
      "id": 3139830397,
      "node_id": "I_kwDOJVxJus67JgJ9",
      "number": 20960,
      "title": "PluginInvokeError No such file or directory",
      "user": {
        "login": "nicho2",
        "id": 11471811,
        "node_id": "MDQ6VXNlcjExNDcxODEx",
        "avatar_url": "https://avatars.githubusercontent.com/u/11471811?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/nicho2",
        "html_url": "https://github.com/nicho2",
        "followers_url": "https://api.github.com/users/nicho2/followers",
        "following_url": "https://api.github.com/users/nicho2/following{/other_user}",
        "gists_url": "https://api.github.com/users/nicho2/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/nicho2/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/nicho2/subscriptions",
        "organizations_url": "https://api.github.com/users/nicho2/orgs",
        "repos_url": "https://api.github.com/users/nicho2/repos",
        "events_url": "https://api.github.com/users/nicho2/events{/privacy}",
        "received_events_url": "https://api.github.com/users/nicho2/received_events",
        "type": "User",
        "user_view_type": "public",
        "site_admin": false
      },
      "labels": [
        {
          "id": 5379590677,
          "node_id": "LA_kwDOJVxJus8AAAABQKYKFQ",
          "url": "https://api.github.com/repos/langgenius/dify/labels/%F0%9F%90%9E%20bug",
          "name": "üêû bug",
          "color": "FFC300",
          "default": false,
          "description": "Something isn't working"
        }
      ],
      "state": "closed",
      "locked": false,
      "assignee": null,
      "assignees": [],
      "milestone": null,
      "comments": 3,
      "created_at": "2025-06-12T11:05:23Z",
      "updated_at": "2025-06-12T13:27:17Z",
      "closed_at": "2025-06-12T13:27:17Z",
      "author_association": "NONE",
      "type": null,
      "active_lock_reason": null,
      "sub_issues_summary": {
        "total": 0,
        "completed": 0,
        "percent_completed": 0
      },
      "body": "### Self Checks\n\n- [x] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).\n- [x] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.\n- [x] I confirm that I am using English to submit this report (ÊàëÂ∑≤ÈòÖËØªÂπ∂ÂêåÊÑè [Language Policy](https://github.com/langgenius/dify/issues/1542)).\n- [x] [FOR CHINESE USERS] ËØ∑Âä°ÂøÖ‰ΩøÁî®Ëã±ÊñáÊèê‰∫§ IssueÔºåÂê¶Âàô‰ºöË¢´ÂÖ≥Èó≠„ÄÇË∞¢Ë∞¢ÔºÅ:)\n- [x] Please do not modify this template :) and fill in all the required fields.\n\n### Dify version\n\n1.4.2\n\n### Cloud or Self Hosted\n\nSelf Hosted (Docker)\n\n### Steps to reproduce\n\nHello,\n\nI just updated form 1.4.0 to version 1.4.2.\nAnd the two plugins, azure_openai and azure_ai_studio, no longer work.\nHowever, the openai and ollama plugins work fine.\n\nazure_openai plugin version: 0.0.16\nazure_ai_studio plugin version: 0.0.3\n\n\n\nTo reproduce, just use a model from a chatbot\n\n\n\n\n\n### ‚úîÔ∏è Expected Behavior\n\nhave no error\n\n### ‚ùå Actual Behavior\n\nFor azure_openai, I get the following error:\n\n[azure_openai] Error: PluginInvokeError: {\"args\":{\"description\":\"[models] Error: [Errno 2] No such file or directory\"},\"error_type\":\"InvokeError\",\"message\":\"[models] Error: [Errno 2] No such file or directory\"}\n\nin log api container :\n\n    [on_llm_new_chunk]2025-06-12 09:18:43.490 ERROR [Thread-1244 (worker_with_context)] [app_generator.py:244] - Error when generating\n    \n    Traceback (most recent call last):\n    \n      File \"/app/api/core/model_runtime/model_providers/__base/large_language_model.py\", line 275, in _invoke_result_generator\n    \n        for chunk in result:\n    \n                     ^^^^^^\n    \n      File \"/app/api/core/plugin/impl/model.py\", line 189, in invoke_llm\n    \n        yield from response\n    \n      File \"/app/api/core/plugin/impl/base.py\", line 211, in _request_with_plugin_daemon_response_stream\n    \n        self._handle_plugin_daemon_error(error.error_type, error.message)\n    \n      File \"/app/api/core/plugin/impl/base.py\", line 245, in _handle_plugin_daemon_error\n    \n        raise PluginInvokeError(description=message)\n    \n    core.plugin.impl.exc.PluginInvokeError: PluginInvokeError: {\"args\":{\"description\":\"[models] Error: [Errno 2] No such file or directory\"},\"error_type\":\"InvokeError\",\"message\":\"[models] Error: [Errno 2] No such file or directory\"}\n    \n    During handling of the above exception, another exception occurred:\n    \n    Traceback (most recent call last):\n    \n      File \"/app/api/core/app/apps/chat/app_generator.py\", line 227, in _generate_worker\n    \n        runner.run(\n    \n      File \"/app/api/core/app/apps/chat/app_runner.py\", line 217, in run\n    \n        self._handle_invoke_result(\n    \n      File \"/app/api/core/app/apps/base_app_runner.py\", line 279, in _handle_invoke_result\n    \n        self._handle_invoke_result_stream(invoke_result=invoke_result, queue_manager=queue_manager, agent=agent)\n    \n      File \"/app/api/core/app/apps/base_app_runner.py\", line 314, in _handle_invoke_result_stream\n    \n        for result in invoke_result:\n    \n                      ^^^^^^^^^^^^^\n    \n      File \"/app/api/core/model_runtime/model_providers/__base/large_language_model.py\", line 304, in _invoke_result_generator\n    \n        raise self._transform_invoke_error(e)\n    \n    core.model_runtime.errors.invoke.InvokeError: [azure_openai] Error: PluginInvokeError: {\"args\":{\"description\":\"[models] Error: [Errno 2] No such file or directory\"},\"error_type\":\"InvokeError\",\"message\":\"[models] Error: [Errno 2] No such file or directory\"}\n    \n    2025-06-12 09:18:43.494 INFO [Thread-1243 (process_request_thread)] [_internal.py:97] - 172.23.0.11 - - [12/Jun/2025 09:18:43] \"POST /console/api/apps/82593598-82d7-4003-b3cd-6a8385a6b649/chat-messages HTTP/1.1\" 200 -\n\nand for azure_ai_studio:\n\n[azure_ai_studio] Error: PluginInvokeError: {\"args\":{\"description\":\"[models] Error: [models] Error: Could not find a suitable TLS CA certificate bundle, invalid path: /usr/local/share/ca-certificates/dify.crt\"},\"error_type\":\"InvokeError\",\"message\":\"[models] Error: [models] Error: Could not find a suitable TLS CA certificate bundle, invalid path: /usr/local/share/ca-certificates/dify.crt\"}\n\n\n    [on_llm_new_chunk]2025-06-12 09:16:49.504 ERROR [Thread-1239 (worker_with_context)] [app_generator.py:244] - Error when generating\n    \n    Traceback (most recent call last):\n    \n      File \"/app/api/core/model_runtime/model_providers/__base/large_language_model.py\", line 275, in _invoke_result_generator\n    \n        for chunk in result:\n    \n                     ^^^^^^\n    \n      File \"/app/api/core/plugin/impl/model.py\", line 189, in invoke_llm\n    \n        yield from response\n    \n      File \"/app/api/core/plugin/impl/base.py\", line 211, in _request_with_plugin_daemon_response_stream\n    \n        self._handle_plugin_daemon_error(error.error_type, error.message)\n    \n      File \"/app/api/core/plugin/impl/base.py\", line 245, in _handle_plugin_daemon_error\n    \n        raise PluginInvokeError(description=message)\n    \n    core.plugin.impl.exc.PluginInvokeError: PluginInvokeError: {\"args\":{\"description\":\"[models] Error: [models] Error: Could not find a suitable TLS CA certificate bundle, invalid path: /usr/local/share/ca-certificates/dify.crt\"},\"error_type\":\"InvokeError\",\"message\":\"[models] Error: [models] Error: Could not find a suitable TLS CA certificate bundle, invalid path: /usr/local/share/ca-certificates/dify.crt\"}\n    \n    During handling of the above exception, another exception occurred:\n    \n    Traceback (most recent call last):\n    \n      File \"/app/api/core/app/apps/chat/app_generator.py\", line 227, in _generate_worker\n    \n        runner.run(\n    \n      File \"/app/api/core/app/apps/chat/app_runner.py\", line 217, in run\n    \n        self._handle_invoke_result(\n    \n      File \"/app/api/core/app/apps/base_app_runner.py\", line 279, in _handle_invoke_result\n    \n        self._handle_invoke_result_stream(invoke_result=invoke_result, queue_manager=queue_manager, agent=agent)\n    \n      File \"/app/api/core/app/apps/base_app_runner.py\", line 314, in _handle_invoke_result_stream\n    \n        for result in invoke_result:\n    \n                      ^^^^^^^^^^^^^\n    \n      File \"/app/api/core/model_runtime/model_providers/__base/large_language_model.py\", line 304, in _invoke_result_generator\n    \n        raise self._transform_invoke_error(e)\n    \n    core.model_runtime.errors.invoke.InvokeError: [azure_ai_studio] Error: PluginInvokeError: {\"args\":{\"description\":\"[models] Error: [models] Error: Could not find a suitable TLS CA certificate bundle, invalid path: /usr/local/share/ca-certificates/dify.crt\"},\"error_type\":\"InvokeError\",\"message\":\"[models] Error: [models] Error: Could not find a suitable TLS CA certificate bundle, invalid path: /usr/local/share/ca-certificates/dify.crt\"}\n    \n    2025-06-12 09:16:49.509 INFO [Thread-1238 (process_request_thread)] [_internal.py:97] - 172.23.0.11 - - [12/Jun/2025 09:16:49] \"POST /console/api/apps/82593598-82d7-4003-b3cd-6a8385a6b649/chat-messages HTTP/1.1\" 200 -\n\n\n",
      "closed_by": {
        "login": "nicho2",
        "id": 11471811,
        "node_id": "MDQ6VXNlcjExNDcxODEx",
        "avatar_url": "https://avatars.githubusercontent.com/u/11471811?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/nicho2",
        "html_url": "https://github.com/nicho2",
        "followers_url": "https://api.github.com/users/nicho2/followers",
        "following_url": "https://api.github.com/users/nicho2/following{/other_user}",
        "gists_url": "https://api.github.com/users/nicho2/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/nicho2/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/nicho2/subscriptions",
        "organizations_url": "https://api.github.com/users/nicho2/orgs",
        "repos_url": "https://api.github.com/users/nicho2/repos",
        "events_url": "https://api.github.com/users/nicho2/events{/privacy}",
        "received_events_url": "https://api.github.com/users/nicho2/received_events",
        "type": "User",
        "user_view_type": "public",
        "site_admin": false
      },
      "reactions": {
        "url": "https://api.github.com/repos/langgenius/dify/issues/20960/reactions",
        "total_count": 1,
        "+1": 1,
        "-1": 0,
        "laugh": 0,
        "hooray": 0,
        "confused": 0,
        "heart": 0,
        "rocket": 0,
        "eyes": 0
      },
      "timeline_url": "https://api.github.com/repos/langgenius/dify/issues/20960/timeline",
      "performed_via_github_app": null,
      "state_reason": "completed"
    },
    {
      "url": "https://api.github.com/repos/langgenius/dify/issues/20957",
      "repository_url": "https://api.github.com/repos/langgenius/dify",
      "labels_url": "https://api.github.com/repos/langgenius/dify/issues/20957/labels{/name}",
      "comments_url": "https://api.github.com/repos/langgenius/dify/issues/20957/comments",
      "events_url": "https://api.github.com/repos/langgenius/dify/issues/20957/events",
      "html_url": "https://github.com/langgenius/dify/pull/20957",
      "id": 3139548452,
      "node_id": "PR_kwDOJVxJus6aLbtz",
      "number": 20957,
      "title": "Add test button to login page",
      "user": {
        "login": "NightGovery",
        "id": 188849281,
        "node_id": "U_kgDOC0GcgQ",
        "avatar_url": "https://avatars.githubusercontent.com/u/188849281?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/NightGovery",
        "html_url": "https://github.com/NightGovery",
        "followers_url": "https://api.github.com/users/NightGovery/followers",
        "following_url": "https://api.github.com/users/NightGovery/following{/other_user}",
        "gists_url": "https://api.github.com/users/NightGovery/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/NightGovery/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/NightGovery/subscriptions",
        "organizations_url": "https://api.github.com/users/NightGovery/orgs",
        "repos_url": "https://api.github.com/users/NightGovery/repos",
        "events_url": "https://api.github.com/users/NightGovery/events{/privacy}",
        "received_events_url": "https://api.github.com/users/NightGovery/received_events",
        "type": "User",
        "user_view_type": "public",
        "site_admin": false
      },
      "labels": [
        {
          "id": 5379590684,
          "node_id": "LA_kwDOJVxJus8AAAABQKYKHA",
          "url": "https://api.github.com/repos/langgenius/dify/labels/%F0%9F%93%9A%20documentation",
          "name": "üìö documentation",
          "color": "0075ca",
          "default": false,
          "description": "Improvements or additions to documentation"
        },
        {
          "id": 6269723800,
          "node_id": "LA_kwDOJVxJus8AAAABdbRkmA",
          "url": "https://api.github.com/repos/langgenius/dify/labels/size:S",
          "name": "size:S",
          "color": "77b800",
          "default": false,
          "description": "This PR changes 10-29 lines, ignoring generated files."
        },
        {
          "id": 6759329484,
          "node_id": "LA_kwDOJVxJus8AAAABkuMuzA",
          "url": "https://api.github.com/repos/langgenius/dify/labels/%E2%98%95%EF%B8%8F%20typescript",
          "name": "‚òïÔ∏è typescript",
          "color": "006b75",
          "default": false,
          "description": "Pull request that update TypeScript code."
        }
      ],
      "state": "closed",
      "locked": false,
      "assignee": null,
      "assignees": [],
      "milestone": null,
      "comments": 1,
      "created_at": "2025-06-12T09:36:09Z",
      "updated_at": "2025-06-12T09:51:06Z",
      "closed_at": "2025-06-12T09:36:38Z",
      "author_association": "NONE",
      "type": null,
      "active_lock_reason": null,
      "draft": false,
      "pull_request": {
        "url": "https://api.github.com/repos/langgenius/dify/pulls/20957",
        "html_url": "https://github.com/langgenius/dify/pull/20957",
        "diff_url": "https://github.com/langgenius/dify/pull/20957.diff",
        "patch_url": "https://github.com/langgenius/dify/pull/20957.patch",
        "merged_at": null
      },
      "body": "> [!IMPORTANT]\r\n>\r\n> 1. Make sure you have read our [contribution guidelines](https://github.com/langgenius/dify/blob/main/CONTRIBUTING.md)\r\n> 2. Ensure there is an associated issue and you have been assigned to it\r\n> 3. Use the correct syntax to link this PR: `Fixes #<issue number>`.\r\n\r\n## Summary\r\n\r\n<!-- Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change. -->\r\n\r\n## Screenshots\r\n\r\n| Before | After |\r\n|--------|-------|\r\n| ...    | ...   |\r\n\r\n## Checklist\r\n\r\n- [ ] This change requires a documentation update, included: [Dify Document](https://github.com/langgenius/dify-docs)\r\n- [x] I understand that this PR may be closed in case there was no previous discussion or issues. (This doesn't apply to typos!)\r\n- [x] I've added a test for each change that was introduced, and I tried as much as possible to make a single atomic change.\r\n- [x] I've updated the documentation accordingly.\r\n- [x] I ran `dev/reformat`(backend) and `cd web && npx lint-staged`(frontend) to appease the lint gods\r\n",
      "closed_by": {
        "login": "NightGovery",
        "id": 188849281,
        "node_id": "U_kgDOC0GcgQ",
        "avatar_url": "https://avatars.githubusercontent.com/u/188849281?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/NightGovery",
        "html_url": "https://github.com/NightGovery",
        "followers_url": "https://api.github.com/users/NightGovery/followers",
        "following_url": "https://api.github.com/users/NightGovery/following{/other_user}",
        "gists_url": "https://api.github.com/users/NightGovery/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/NightGovery/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/NightGovery/subscriptions",
        "organizations_url": "https://api.github.com/users/NightGovery/orgs",
        "repos_url": "https://api.github.com/users/NightGovery/repos",
        "events_url": "https://api.github.com/users/NightGovery/events{/privacy}",
        "received_events_url": "https://api.github.com/users/NightGovery/received_events",
        "type": "User",
        "user_view_type": "public",
        "site_admin": false
      },
      "reactions": {
        "url": "https://api.github.com/repos/langgenius/dify/issues/20957/reactions",
        "total_count": 0,
        "+1": 0,
        "-1": 0,
        "laugh": 0,
        "hooray": 0,
        "confused": 0,
        "heart": 0,
        "rocket": 0,
        "eyes": 0
      },
      "timeline_url": "https://api.github.com/repos/langgenius/dify/issues/20957/timeline",
      "performed_via_github_app": null,
      "state_reason": null
    },
    {
      "url": "https://api.github.com/repos/langgenius/dify/issues/20951",
      "repository_url": "https://api.github.com/repos/langgenius/dify",
      "labels_url": "https://api.github.com/repos/langgenius/dify/issues/20951/labels{/name}",
      "comments_url": "https://api.github.com/repos/langgenius/dify/issues/20951/comments",
      "events_url": "https://api.github.com/repos/langgenius/dify/issues/20951/events",
      "html_url": "https://github.com/langgenius/dify/issues/20951",
      "id": 3139364105,
      "node_id": "I_kwDOJVxJus67HuUJ",
      "number": 20951,
      "title": "chatflow    Direct output cannot display image",
      "user": {
        "login": "jiayi-1994",
        "id": 120161630,
        "node_id": "U_kgDOBymFXg",
        "avatar_url": "https://avatars.githubusercontent.com/u/120161630?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/jiayi-1994",
        "html_url": "https://github.com/jiayi-1994",
        "followers_url": "https://api.github.com/users/jiayi-1994/followers",
        "following_url": "https://api.github.com/users/jiayi-1994/following{/other_user}",
        "gists_url": "https://api.github.com/users/jiayi-1994/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/jiayi-1994/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/jiayi-1994/subscriptions",
        "organizations_url": "https://api.github.com/users/jiayi-1994/orgs",
        "repos_url": "https://api.github.com/users/jiayi-1994/repos",
        "events_url": "https://api.github.com/users/jiayi-1994/events{/privacy}",
        "received_events_url": "https://api.github.com/users/jiayi-1994/received_events",
        "type": "User",
        "user_view_type": "public",
        "site_admin": false
      },
      "labels": [],
      "state": "closed",
      "locked": false,
      "assignee": null,
      "assignees": [],
      "milestone": null,
      "comments": 2,
      "created_at": "2025-06-12T08:35:34Z",
      "updated_at": "2025-06-12T09:06:16Z",
      "closed_at": "2025-06-12T09:06:16Z",
      "author_association": "NONE",
      "type": null,
      "active_lock_reason": null,
      "sub_issues_summary": {
        "total": 0,
        "completed": 0,
        "percent_completed": 0
      },
      "body": "### Self Checks\n\n- [x] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).\n- [x] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.\n- [x] I confirm that I am using English to submit this report (ÊàëÂ∑≤ÈòÖËØªÂπ∂ÂêåÊÑè [Language Policy](https://github.com/langgenius/dify/issues/1542)).\n- [x] [FOR CHINESE USERS] ËØ∑Âä°ÂøÖ‰ΩøÁî®Ëã±ÊñáÊèê‰∫§ IssueÔºåÂê¶Âàô‰ºöË¢´ÂÖ≥Èó≠„ÄÇË∞¢Ë∞¢ÔºÅ:)\n- [x] Please do not modify this template :) and fill in all the required fields.\n\n### Dify version\n\n1.4.1\n\n### Cloud or Self Hosted\n\nSelf Hosted (Docker)\n\n### Steps to reproduce\n\n![Image](https://github.com/user-attachments/assets/c8cbf7de-3306-41aa-bb14-a4d43f439c3f)\n\n### ‚úîÔ∏è Expected Behavior\n\ncan use html or markdown?\n\n### ‚ùå Actual Behavior\n\n_No response_",
      "closed_by": {
        "login": "jiayi-1994",
        "id": 120161630,
        "node_id": "U_kgDOBymFXg",
        "avatar_url": "https://avatars.githubusercontent.com/u/120161630?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/jiayi-1994",
        "html_url": "https://github.com/jiayi-1994",
        "followers_url": "https://api.github.com/users/jiayi-1994/followers",
        "following_url": "https://api.github.com/users/jiayi-1994/following{/other_user}",
        "gists_url": "https://api.github.com/users/jiayi-1994/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/jiayi-1994/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/jiayi-1994/subscriptions",
        "organizations_url": "https://api.github.com/users/jiayi-1994/orgs",
        "repos_url": "https://api.github.com/users/jiayi-1994/repos",
        "events_url": "https://api.github.com/users/jiayi-1994/events{/privacy}",
        "received_events_url": "https://api.github.com/users/jiayi-1994/received_events",
        "type": "User",
        "user_view_type": "public",
        "site_admin": false
      },
      "reactions": {
        "url": "https://api.github.com/repos/langgenius/dify/issues/20951/reactions",
        "total_count": 1,
        "+1": 1,
        "-1": 0,
        "laugh": 0,
        "hooray": 0,
        "confused": 0,
        "heart": 0,
        "rocket": 0,
        "eyes": 0
      },
      "timeline_url": "https://api.github.com/repos/langgenius/dify/issues/20951/timeline",
      "performed_via_github_app": null,
      "state_reason": "completed"
    },
    {
      "url": "https://api.github.com/repos/langgenius/dify/issues/20933",
      "repository_url": "https://api.github.com/repos/langgenius/dify",
      "labels_url": "https://api.github.com/repos/langgenius/dify/issues/20933/labels{/name}",
      "comments_url": "https://api.github.com/repos/langgenius/dify/issues/20933/comments",
      "events_url": "https://api.github.com/repos/langgenius/dify/issues/20933/events",
      "html_url": "https://github.com/langgenius/dify/pull/20933",
      "id": 3138851694,
      "node_id": "PR_kwDOJVxJus6aJEKn",
      "number": 20933,
      "title": "fix: markdown button can't send message",
      "user": {
        "login": "hjlarry",
        "id": 25834719,
        "node_id": "MDQ6VXNlcjI1ODM0NzE5",
        "avatar_url": "https://avatars.githubusercontent.com/u/25834719?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/hjlarry",
        "html_url": "https://github.com/hjlarry",
        "followers_url": "https://api.github.com/users/hjlarry/followers",
        "following_url": "https://api.github.com/users/hjlarry/following{/other_user}",
        "gists_url": "https://api.github.com/users/hjlarry/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/hjlarry/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/hjlarry/subscriptions",
        "organizations_url": "https://api.github.com/users/hjlarry/orgs",
        "repos_url": "https://api.github.com/users/hjlarry/repos",
        "events_url": "https://api.github.com/users/hjlarry/events{/privacy}",
        "received_events_url": "https://api.github.com/users/hjlarry/received_events",
        "type": "User",
        "user_view_type": "public",
        "site_admin": false
      },
      "labels": [
        {
          "id": 6269723781,
          "node_id": "LA_kwDOJVxJus8AAAABdbRkhQ",
          "url": "https://api.github.com/repos/langgenius/dify/labels/size:XS",
          "name": "size:XS",
          "color": "00ff00",
          "default": false,
          "description": "This PR changes 0-9 lines, ignoring generated files."
        },
        {
          "id": 6756620282,
          "node_id": "LA_kwDOJVxJus8AAAABkrnX-g",
          "url": "https://api.github.com/repos/langgenius/dify/labels/lgtm",
          "name": "lgtm",
          "color": "238636",
          "default": false,
          "description": "This PR has been approved by a maintainer"
        },
        {
          "id": 6759329484,
          "node_id": "LA_kwDOJVxJus8AAAABkuMuzA",
          "url": "https://api.github.com/repos/langgenius/dify/labels/%E2%98%95%EF%B8%8F%20typescript",
          "name": "‚òïÔ∏è typescript",
          "color": "006b75",
          "default": false,
          "description": "Pull request that update TypeScript code."
        }
      ],
      "state": "closed",
      "locked": false,
      "assignee": null,
      "assignees": [],
      "milestone": null,
      "comments": 0,
      "created_at": "2025-06-12T05:02:06Z",
      "updated_at": "2025-06-12T06:18:16Z",
      "closed_at": "2025-06-12T06:18:15Z",
      "author_association": "CONTRIBUTOR",
      "type": null,
      "active_lock_reason": null,
      "draft": false,
      "pull_request": {
        "url": "https://api.github.com/repos/langgenius/dify/pulls/20933",
        "html_url": "https://github.com/langgenius/dify/pull/20933",
        "diff_url": "https://github.com/langgenius/dify/pull/20933.diff",
        "patch_url": "https://github.com/langgenius/dify/pull/20933.patch",
        "merged_at": "2025-06-12T06:18:15Z"
      },
      "body": "> [!IMPORTANT]\r\n>\r\n> 1. Make sure you have read our [contribution guidelines](https://github.com/langgenius/dify/blob/main/CONTRIBUTING.md)\r\n> 2. Ensure there is an associated issue and you have been assigned to it\r\n> 3. Use the correct syntax to link this PR: `Fixes #<issue number>`.\r\n\r\n## Summary\r\n\r\n<!-- Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change. -->\r\n\r\nfix https://github.com/langgenius/dify/issues/20932\r\n\r\ncaused by https://github.com/langgenius/dify/pull/20639\r\n\r\n![dd74463a58e984814e0e93f1ebe4afa8](https://github.com/user-attachments/assets/e4f69263-d7ae-461f-96a8-9414f4214a61)\r\n\r\n\r\n## Screenshots\r\n\r\n| Before | After |\r\n|--------|-------|\r\n| ...    | ...   |\r\n\r\n## Checklist\r\n\r\n- [ ] This change requires a documentation update, included: [Dify Document](https://github.com/langgenius/dify-docs)\r\n- [x] I understand that this PR may be closed in case there was no previous discussion or issues. (This doesn't apply to typos!)\r\n- [x] I've added a test for each change that was introduced, and I tried as much as possible to make a single atomic change.\r\n- [x] I've updated the documentation accordingly.\r\n- [x] I ran `dev/reformat`(backend) and `cd web && npx lint-staged`(frontend) to appease the lint gods\r\n",
      "closed_by": {
        "login": "crazywoola",
        "id": 100913391,
        "node_id": "U_kgDOBgPQ7w",
        "avatar_url": "https://avatars.githubusercontent.com/u/100913391?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/crazywoola",
        "html_url": "https://github.com/crazywoola",
        "followers_url": "https://api.github.com/users/crazywoola/followers",
        "following_url": "https://api.github.com/users/crazywoola/following{/other_user}",
        "gists_url": "https://api.github.com/users/crazywoola/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/crazywoola/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/crazywoola/subscriptions",
        "organizations_url": "https://api.github.com/users/crazywoola/orgs",
        "repos_url": "https://api.github.com/users/crazywoola/repos",
        "events_url": "https://api.github.com/users/crazywoola/events{/privacy}",
        "received_events_url": "https://api.github.com/users/crazywoola/received_events",
        "type": "User",
        "user_view_type": "public",
        "site_admin": false
      },
      "reactions": {
        "url": "https://api.github.com/repos/langgenius/dify/issues/20933/reactions",
        "total_count": 1,
        "+1": 1,
        "-1": 0,
        "laugh": 0,
        "hooray": 0,
        "confused": 0,
        "heart": 0,
        "rocket": 0,
        "eyes": 0
      },
      "timeline_url": "https://api.github.com/repos/langgenius/dify/issues/20933/timeline",
      "performed_via_github_app": null,
      "state_reason": null
    },
    {
      "url": "https://api.github.com/repos/langgenius/dify/issues/20932",
      "repository_url": "https://api.github.com/repos/langgenius/dify",
      "labels_url": "https://api.github.com/repos/langgenius/dify/issues/20932/labels{/name}",
      "comments_url": "https://api.github.com/repos/langgenius/dify/issues/20932/comments",
      "events_url": "https://api.github.com/repos/langgenius/dify/issues/20932/events",
      "html_url": "https://github.com/langgenius/dify/issues/20932",
      "id": 3138697750,
      "node_id": "I_kwDOJVxJus67FLoW",
      "number": 20932,
      "title": "Tapping the added button tag does not send a message.",
      "user": {
        "login": "Miranofuu-Dodoria",
        "id": 210794098,
        "node_id": "U_kgDODJB2cg",
        "avatar_url": "https://avatars.githubusercontent.com/u/210794098?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/Miranofuu-Dodoria",
        "html_url": "https://github.com/Miranofuu-Dodoria",
        "followers_url": "https://api.github.com/users/Miranofuu-Dodoria/followers",
        "following_url": "https://api.github.com/users/Miranofuu-Dodoria/following{/other_user}",
        "gists_url": "https://api.github.com/users/Miranofuu-Dodoria/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/Miranofuu-Dodoria/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/Miranofuu-Dodoria/subscriptions",
        "organizations_url": "https://api.github.com/users/Miranofuu-Dodoria/orgs",
        "repos_url": "https://api.github.com/users/Miranofuu-Dodoria/repos",
        "events_url": "https://api.github.com/users/Miranofuu-Dodoria/events{/privacy}",
        "received_events_url": "https://api.github.com/users/Miranofuu-Dodoria/received_events",
        "type": "User",
        "user_view_type": "public",
        "site_admin": false
      },
      "labels": [
        {
          "id": 5379590677,
          "node_id": "LA_kwDOJVxJus8AAAABQKYKFQ",
          "url": "https://api.github.com/repos/langgenius/dify/labels/%F0%9F%90%9E%20bug",
          "name": "üêû bug",
          "color": "FFC300",
          "default": false,
          "description": "Something isn't working"
        }
      ],
      "state": "closed",
      "locked": false,
      "assignee": null,
      "assignees": [],
      "milestone": null,
      "comments": 1,
      "created_at": "2025-06-12T03:27:22Z",
      "updated_at": "2025-06-12T06:18:16Z",
      "closed_at": "2025-06-12T06:18:16Z",
      "author_association": "NONE",
      "type": null,
      "active_lock_reason": null,
      "sub_issues_summary": {
        "total": 0,
        "completed": 0,
        "percent_completed": 0
      },
      "body": "### Self Checks\n\n- [x] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).\n- [x] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.\n- [x] I confirm that I am using English to submit this report (ÊàëÂ∑≤ÈòÖËØªÂπ∂ÂêåÊÑè [Language Policy](https://github.com/langgenius/dify/issues/1542)).\n- [x] [FOR CHINESE USERS] ËØ∑Âä°ÂøÖ‰ΩøÁî®Ëã±ÊñáÊèê‰∫§ IssueÔºåÂê¶Âàô‰ºöË¢´ÂÖ≥Èó≠„ÄÇË∞¢Ë∞¢ÔºÅ:)\n- [x] Please do not modify this template :) and fill in all the required fields.\n\n### Dify version\n\n1.4.2\n\n### Cloud or Self Hosted\n\nCloud\n\n### Steps to reproduce\n\nfeat: support button in markdown #9876, the content described in the data-message cannot be sent after tapping the button.\n\nUntil v1.4.1, the message was sent after the button was tapped, but as of v1.4.2, the message is not sent even if the button is tapped.\n\n### ‚úîÔ∏è Expected Behavior\n\nMessage is sent after tapping the button as in v1.4.1.\n\n### ‚ùå Actual Behavior\n\nfeat: support button in markdown #9876 and not the same behavior.",
      "closed_by": {
        "login": "crazywoola",
        "id": 100913391,
        "node_id": "U_kgDOBgPQ7w",
        "avatar_url": "https://avatars.githubusercontent.com/u/100913391?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/crazywoola",
        "html_url": "https://github.com/crazywoola",
        "followers_url": "https://api.github.com/users/crazywoola/followers",
        "following_url": "https://api.github.com/users/crazywoola/following{/other_user}",
        "gists_url": "https://api.github.com/users/crazywoola/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/crazywoola/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/crazywoola/subscriptions",
        "organizations_url": "https://api.github.com/users/crazywoola/orgs",
        "repos_url": "https://api.github.com/users/crazywoola/repos",
        "events_url": "https://api.github.com/users/crazywoola/events{/privacy}",
        "received_events_url": "https://api.github.com/users/crazywoola/received_events",
        "type": "User",
        "user_view_type": "public",
        "site_admin": false
      },
      "reactions": {
        "url": "https://api.github.com/repos/langgenius/dify/issues/20932/reactions",
        "total_count": 2,
        "+1": 2,
        "-1": 0,
        "laugh": 0,
        "hooray": 0,
        "confused": 0,
        "heart": 0,
        "rocket": 0,
        "eyes": 0
      },
      "timeline_url": "https://api.github.com/repos/langgenius/dify/issues/20932/timeline",
      "performed_via_github_app": null,
      "state_reason": "completed"
    },
    {
      "url": "https://api.github.com/repos/langgenius/dify/issues/20929",
      "repository_url": "https://api.github.com/repos/langgenius/dify",
      "labels_url": "https://api.github.com/repos/langgenius/dify/issues/20929/labels{/name}",
      "comments_url": "https://api.github.com/repos/langgenius/dify/issues/20929/comments",
      "events_url": "https://api.github.com/repos/langgenius/dify/issues/20929/events",
      "html_url": "https://github.com/langgenius/dify/pull/20929",
      "id": 3138596250,
      "node_id": "PR_kwDOJVxJus6aINgp",
      "number": 20929,
      "title": "Develop",
      "user": {
        "login": "ynakazawa-cx",
        "id": 214411530,
        "node_id": "U_kgDODMepCg",
        "avatar_url": "https://avatars.githubusercontent.com/u/214411530?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/ynakazawa-cx",
        "html_url": "https://github.com/ynakazawa-cx",
        "followers_url": "https://api.github.com/users/ynakazawa-cx/followers",
        "following_url": "https://api.github.com/users/ynakazawa-cx/following{/other_user}",
        "gists_url": "https://api.github.com/users/ynakazawa-cx/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/ynakazawa-cx/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/ynakazawa-cx/subscriptions",
        "organizations_url": "https://api.github.com/users/ynakazawa-cx/orgs",
        "repos_url": "https://api.github.com/users/ynakazawa-cx/repos",
        "events_url": "https://api.github.com/users/ynakazawa-cx/events{/privacy}",
        "received_events_url": "https://api.github.com/users/ynakazawa-cx/received_events",
        "type": "User",
        "user_view_type": "public",
        "site_admin": false
      },
      "labels": [
        {
          "id": 5379590684,
          "node_id": "LA_kwDOJVxJus8AAAABQKYKHA",
          "url": "https://api.github.com/repos/langgenius/dify/labels/%F0%9F%93%9A%20documentation",
          "name": "üìö documentation",
          "color": "0075ca",
          "default": false,
          "description": "Improvements or additions to documentation"
        },
        {
          "id": 6269723843,
          "node_id": "LA_kwDOJVxJus8AAAABdbRkww",
          "url": "https://api.github.com/repos/langgenius/dify/labels/size:L",
          "name": "size:L",
          "color": "eb9500",
          "default": false,
          "description": "This PR changes 100-499 lines, ignoring generated files."
        }
      ],
      "state": "closed",
      "locked": false,
      "assignee": null,
      "assignees": [],
      "milestone": null,
      "comments": 0,
      "created_at": "2025-06-12T02:33:01Z",
      "updated_at": "2025-06-12T06:18:30Z",
      "closed_at": "2025-06-12T06:18:30Z",
      "author_association": "NONE",
      "type": null,
      "active_lock_reason": null,
      "draft": false,
      "pull_request": {
        "url": "https://api.github.com/repos/langgenius/dify/pulls/20929",
        "html_url": "https://github.com/langgenius/dify/pull/20929",
        "diff_url": "https://github.com/langgenius/dify/pull/20929.diff",
        "patch_url": "https://github.com/langgenius/dify/pull/20929.patch",
        "merged_at": null
      },
      "body": "> [!IMPORTANT]\r\n>\r\n> 1. Make sure you have read our [contribution guidelines](https://github.com/langgenius/dify/blob/main/CONTRIBUTING.md)\r\n> 2. Ensure there is an associated issue and you have been assigned to it\r\n> 3. Use the correct syntax to link this PR: `Fixes #<issue number>`.\r\n\r\n## Summary\r\n\r\n<!-- Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change. -->\r\n\r\n## Screenshots\r\n\r\n| Before | After |\r\n|--------|-------|\r\n| ...    | ...   |\r\n\r\n## Checklist\r\n\r\n- [ ] This change requires a documentation update, included: [Dify Document](https://github.com/langgenius/dify-docs)\r\n- [x] I understand that this PR may be closed in case there was no previous discussion or issues. (This doesn't apply to typos!)\r\n- [x] I've added a test for each change that was introduced, and I tried as much as possible to make a single atomic change.\r\n- [x] I've updated the documentation accordingly.\r\n- [x] I ran `dev/reformat`(backend) and `cd web && npx lint-staged`(frontend) to appease the lint gods\r\n",
      "closed_by": {
        "login": "crazywoola",
        "id": 100913391,
        "node_id": "U_kgDOBgPQ7w",
        "avatar_url": "https://avatars.githubusercontent.com/u/100913391?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/crazywoola",
        "html_url": "https://github.com/crazywoola",
        "followers_url": "https://api.github.com/users/crazywoola/followers",
        "following_url": "https://api.github.com/users/crazywoola/following{/other_user}",
        "gists_url": "https://api.github.com/users/crazywoola/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/crazywoola/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/crazywoola/subscriptions",
        "organizations_url": "https://api.github.com/users/crazywoola/orgs",
        "repos_url": "https://api.github.com/users/crazywoola/repos",
        "events_url": "https://api.github.com/users/crazywoola/events{/privacy}",
        "received_events_url": "https://api.github.com/users/crazywoola/received_events",
        "type": "User",
        "user_view_type": "public",
        "site_admin": false
      },
      "reactions": {
        "url": "https://api.github.com/repos/langgenius/dify/issues/20929/reactions",
        "total_count": 0,
        "+1": 0,
        "-1": 0,
        "laugh": 0,
        "hooray": 0,
        "confused": 0,
        "heart": 0,
        "rocket": 0,
        "eyes": 0
      },
      "timeline_url": "https://api.github.com/repos/langgenius/dify/issues/20929/timeline",
      "performed_via_github_app": null,
      "state_reason": null
    },
    {
      "url": "https://api.github.com/repos/langgenius/dify/issues/20913",
      "repository_url": "https://api.github.com/repos/langgenius/dify",
      "labels_url": "https://api.github.com/repos/langgenius/dify/issues/20913/labels{/name}",
      "comments_url": "https://api.github.com/repos/langgenius/dify/issues/20913/comments",
      "events_url": "https://api.github.com/repos/langgenius/dify/issues/20913/events",
      "html_url": "https://github.com/langgenius/dify/issues/20913",
      "id": 3136178623,
      "node_id": "I_kwDOJVxJus667km_",
      "number": 20913,
      "title": "Git Error: \"fatal: not a git repository\" During Dify Docker Update",
      "user": {
        "login": "wangguichao",
        "id": 20470703,
        "node_id": "MDQ6VXNlcjIwNDcwNzAz",
        "avatar_url": "https://avatars.githubusercontent.com/u/20470703?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/wangguichao",
        "html_url": "https://github.com/wangguichao",
        "followers_url": "https://api.github.com/users/wangguichao/followers",
        "following_url": "https://api.github.com/users/wangguichao/following{/other_user}",
        "gists_url": "https://api.github.com/users/wangguichao/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/wangguichao/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/wangguichao/subscriptions",
        "organizations_url": "https://api.github.com/users/wangguichao/orgs",
        "repos_url": "https://api.github.com/users/wangguichao/repos",
        "events_url": "https://api.github.com/users/wangguichao/events{/privacy}",
        "received_events_url": "https://api.github.com/users/wangguichao/received_events",
        "type": "User",
        "user_view_type": "public",
        "site_admin": false
      },
      "labels": [],
      "state": "closed",
      "locked": false,
      "assignee": null,
      "assignees": [],
      "milestone": null,
      "comments": 1,
      "created_at": "2025-06-11T09:43:13Z",
      "updated_at": "2025-06-12T06:14:21Z",
      "closed_at": "2025-06-12T06:14:21Z",
      "author_association": "NONE",
      "type": null,
      "active_lock_reason": null,
      "sub_issues_summary": {
        "total": 0,
        "completed": 0,
        "percent_completed": 0
      },
      "body": "### Self Checks\n\n- [x] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).\n- [x] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.\n- [x] I confirm that I am using English to submit this report (ÊàëÂ∑≤ÈòÖËØªÂπ∂ÂêåÊÑè [Language Policy](https://github.com/langgenius/dify/issues/1542)).\n- [x] [FOR CHINESE USERS] ËØ∑Âä°ÂøÖ‰ΩøÁî®Ëã±ÊñáÊèê‰∫§ IssueÔºåÂê¶Âàô‰ºöË¢´ÂÖ≥Èó≠„ÄÇË∞¢Ë∞¢ÔºÅ:)\n- [x] Please do not modify this template :) and fill in all the required fields.\n\n### Dify version\n\n0.15.3 and Version 1.1.3\n\n### Cloud or Self Hosted\n\nSelf Hosted (Docker)\n\n### Steps to reproduce\n\n![Image](https://github.com/user-attachments/assets/08ca293d-5d07-496c-bbf6-73eb314824fa)\n\nWhile updating Dify, after entering the Docker directory, the following error message appeared:\ndify\\docker> git checkout main\nfatal: not a git repository (or any of the parent directories): .git\n\n### ‚úîÔ∏è Expected Behavior\n\nWhat's the best way to upgrade Dify?How can I solve this problem?\n\n### ‚ùå Actual Behavior\n\n_No response_",
      "closed_by": {
        "login": "crazywoola",
        "id": 100913391,
        "node_id": "U_kgDOBgPQ7w",
        "avatar_url": "https://avatars.githubusercontent.com/u/100913391?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/crazywoola",
        "html_url": "https://github.com/crazywoola",
        "followers_url": "https://api.github.com/users/crazywoola/followers",
        "following_url": "https://api.github.com/users/crazywoola/following{/other_user}",
        "gists_url": "https://api.github.com/users/crazywoola/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/crazywoola/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/crazywoola/subscriptions",
        "organizations_url": "https://api.github.com/users/crazywoola/orgs",
        "repos_url": "https://api.github.com/users/crazywoola/repos",
        "events_url": "https://api.github.com/users/crazywoola/events{/privacy}",
        "received_events_url": "https://api.github.com/users/crazywoola/received_events",
        "type": "User",
        "user_view_type": "public",
        "site_admin": false
      },
      "reactions": {
        "url": "https://api.github.com/repos/langgenius/dify/issues/20913/reactions",
        "total_count": 1,
        "+1": 1,
        "-1": 0,
        "laugh": 0,
        "hooray": 0,
        "confused": 0,
        "heart": 0,
        "rocket": 0,
        "eyes": 0
      },
      "timeline_url": "https://api.github.com/repos/langgenius/dify/issues/20913/timeline",
      "performed_via_github_app": null,
      "state_reason": "completed"
    },
    {
      "url": "https://api.github.com/repos/langgenius/dify/issues/20912",
      "repository_url": "https://api.github.com/repos/langgenius/dify",
      "labels_url": "https://api.github.com/repos/langgenius/dify/issues/20912/labels{/name}",
      "comments_url": "https://api.github.com/repos/langgenius/dify/issues/20912/comments",
      "events_url": "https://api.github.com/repos/langgenius/dify/issues/20912/events",
      "html_url": "https://github.com/langgenius/dify/issues/20912",
      "id": 3136104758,
      "node_id": "I_kwDOJVxJus667Sk2",
      "number": 20912,
      "title": "Structured Output broken after Release 1.4.2",
      "user": {
        "login": "Rodrigo58174",
        "id": 58174750,
        "node_id": "MDQ6VXNlcjU4MTc0NzUw",
        "avatar_url": "https://avatars.githubusercontent.com/u/58174750?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/Rodrigo58174",
        "html_url": "https://github.com/Rodrigo58174",
        "followers_url": "https://api.github.com/users/Rodrigo58174/followers",
        "following_url": "https://api.github.com/users/Rodrigo58174/following{/other_user}",
        "gists_url": "https://api.github.com/users/Rodrigo58174/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/Rodrigo58174/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/Rodrigo58174/subscriptions",
        "organizations_url": "https://api.github.com/users/Rodrigo58174/orgs",
        "repos_url": "https://api.github.com/users/Rodrigo58174/repos",
        "events_url": "https://api.github.com/users/Rodrigo58174/events{/privacy}",
        "received_events_url": "https://api.github.com/users/Rodrigo58174/received_events",
        "type": "User",
        "user_view_type": "public",
        "site_admin": false
      },
      "labels": [
        {
          "id": 5379590677,
          "node_id": "LA_kwDOJVxJus8AAAABQKYKFQ",
          "url": "https://api.github.com/repos/langgenius/dify/labels/%F0%9F%90%9E%20bug",
          "name": "üêû bug",
          "color": "FFC300",
          "default": false,
          "description": "Something isn't working"
        }
      ],
      "state": "closed",
      "locked": false,
      "assignee": {
        "login": "laipz8200",
        "id": 16485841,
        "node_id": "MDQ6VXNlcjE2NDg1ODQx",
        "avatar_url": "https://avatars.githubusercontent.com/u/16485841?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/laipz8200",
        "html_url": "https://github.com/laipz8200",
        "followers_url": "https://api.github.com/users/laipz8200/followers",
        "following_url": "https://api.github.com/users/laipz8200/following{/other_user}",
        "gists_url": "https://api.github.com/users/laipz8200/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/laipz8200/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/laipz8200/subscriptions",
        "organizations_url": "https://api.github.com/users/laipz8200/orgs",
        "repos_url": "https://api.github.com/users/laipz8200/repos",
        "events_url": "https://api.github.com/users/laipz8200/events{/privacy}",
        "received_events_url": "https://api.github.com/users/laipz8200/received_events",
        "type": "User",
        "user_view_type": "public",
        "site_admin": false
      },
      "assignees": [
        {
          "login": "laipz8200",
          "id": 16485841,
          "node_id": "MDQ6VXNlcjE2NDg1ODQx",
          "avatar_url": "https://avatars.githubusercontent.com/u/16485841?v=4",
          "gravatar_id": "",
          "url": "https://api.github.com/users/laipz8200",
          "html_url": "https://github.com/laipz8200",
          "followers_url": "https://api.github.com/users/laipz8200/followers",
          "following_url": "https://api.github.com/users/laipz8200/following{/other_user}",
          "gists_url": "https://api.github.com/users/laipz8200/gists{/gist_id}",
          "starred_url": "https://api.github.com/users/laipz8200/starred{/owner}{/repo}",
          "subscriptions_url": "https://api.github.com/users/laipz8200/subscriptions",
          "organizations_url": "https://api.github.com/users/laipz8200/orgs",
          "repos_url": "https://api.github.com/users/laipz8200/repos",
          "events_url": "https://api.github.com/users/laipz8200/events{/privacy}",
          "received_events_url": "https://api.github.com/users/laipz8200/received_events",
          "type": "User",
          "user_view_type": "public",
          "site_admin": false
        }
      ],
      "milestone": null,
      "comments": 9,
      "created_at": "2025-06-11T09:18:52Z",
      "updated_at": "2025-06-12T04:54:27Z",
      "closed_at": "2025-06-11T10:56:08Z",
      "author_association": "NONE",
      "type": null,
      "active_lock_reason": null,
      "sub_issues_summary": {
        "total": 0,
        "completed": 0,
        "percent_completed": 0
      },
      "body": "### Self Checks\n\n- [x] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).\n- [x] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.\n- [x] I confirm that I am using English to submit this report (ÊàëÂ∑≤ÈòÖËØªÂπ∂ÂêåÊÑè [Language Policy](https://github.com/langgenius/dify/issues/1542)).\n- [x] [FOR CHINESE USERS] ËØ∑Âä°ÂøÖ‰ΩøÁî®Ëã±ÊñáÊèê‰∫§ IssueÔºåÂê¶Âàô‰ºöË¢´ÂÖ≥Èó≠„ÄÇË∞¢Ë∞¢ÔºÅ:)\n- [x] Please do not modify this template :) and fill in all the required fields.\n\n### Dify version\n\n1.4.2\n\n### Cloud or Self Hosted\n\nCloud\n\n### Steps to reproduce\n\nWith the new release, structured output from LLM node stopped working.\n\n### ‚úîÔ∏è Expected Behavior\n\nClean and structured output (image of output prior to release)\n \n\n<img width=\"448\" alt=\"Image\" src=\"https://github.com/user-attachments/assets/ed21886e-c222-4a1a-b92f-02633aaeb476\" />\n\n### ‚ùå Actual Behavior\n\nNot really a clean structured output (image of same node output after release)\n\n<img width=\"453\" alt=\"Image\" src=\"https://github.com/user-attachments/assets/2941e778-53c5-415d-8347-d496ecbec033\" />",
      "closed_by": {
        "login": "laipz8200",
        "id": 16485841,
        "node_id": "MDQ6VXNlcjE2NDg1ODQx",
        "avatar_url": "https://avatars.githubusercontent.com/u/16485841?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/laipz8200",
        "html_url": "https://github.com/laipz8200",
        "followers_url": "https://api.github.com/users/laipz8200/followers",
        "following_url": "https://api.github.com/users/laipz8200/following{/other_user}",
        "gists_url": "https://api.github.com/users/laipz8200/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/laipz8200/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/laipz8200/subscriptions",
        "organizations_url": "https://api.github.com/users/laipz8200/orgs",
        "repos_url": "https://api.github.com/users/laipz8200/repos",
        "events_url": "https://api.github.com/users/laipz8200/events{/privacy}",
        "received_events_url": "https://api.github.com/users/laipz8200/received_events",
        "type": "User",
        "user_view_type": "public",
        "site_admin": false
      },
      "reactions": {
        "url": "https://api.github.com/repos/langgenius/dify/issues/20912/reactions",
        "total_count": 1,
        "+1": 1,
        "-1": 0,
        "laugh": 0,
        "hooray": 0,
        "confused": 0,
        "heart": 0,
        "rocket": 0,
        "eyes": 0
      },
      "timeline_url": "https://api.github.com/repos/langgenius/dify/issues/20912/timeline",
      "performed_via_github_app": null,
      "state_reason": "completed"
    },
    {
      "url": "https://api.github.com/repos/langgenius/dify/issues/20911",
      "repository_url": "https://api.github.com/repos/langgenius/dify",
      "labels_url": "https://api.github.com/repos/langgenius/dify/issues/20911/labels{/name}",
      "comments_url": "https://api.github.com/repos/langgenius/dify/issues/20911/comments",
      "events_url": "https://api.github.com/repos/langgenius/dify/issues/20911/events",
      "html_url": "https://github.com/langgenius/dify/issues/20911",
      "id": 3136091428,
      "node_id": "I_kwDOJVxJus667PUk",
      "number": 20911,
      "title": "there is no input parameter in comfyui  when public as tool  with image as input parameter",
      "user": {
        "login": "Jonathon-jin",
        "id": 174238636,
        "node_id": "U_kgDOCmKrrA",
        "avatar_url": "https://avatars.githubusercontent.com/u/174238636?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/Jonathon-jin",
        "html_url": "https://github.com/Jonathon-jin",
        "followers_url": "https://api.github.com/users/Jonathon-jin/followers",
        "following_url": "https://api.github.com/users/Jonathon-jin/following{/other_user}",
        "gists_url": "https://api.github.com/users/Jonathon-jin/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/Jonathon-jin/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/Jonathon-jin/subscriptions",
        "organizations_url": "https://api.github.com/users/Jonathon-jin/orgs",
        "repos_url": "https://api.github.com/users/Jonathon-jin/repos",
        "events_url": "https://api.github.com/users/Jonathon-jin/events{/privacy}",
        "received_events_url": "https://api.github.com/users/Jonathon-jin/received_events",
        "type": "User",
        "user_view_type": "public",
        "site_admin": false
      },
      "labels": [
        {
          "id": 6470224171,
          "node_id": "LA_kwDOJVxJus8AAAABgafJKw",
          "url": "https://api.github.com/repos/langgenius/dify/labels/%F0%9F%94%A8%20feat:tools",
          "name": "üî® feat:tools",
          "color": "7346DC",
          "default": false,
          "description": "Tools for agent, function call related stuff."
        }
      ],
      "state": "closed",
      "locked": false,
      "assignee": null,
      "assignees": [],
      "milestone": null,
      "comments": 3,
      "created_at": "2025-06-11T09:15:22Z",
      "updated_at": "2025-06-12T05:55:36Z",
      "closed_at": "2025-06-12T05:55:36Z",
      "author_association": "NONE",
      "type": null,
      "active_lock_reason": null,
      "sub_issues_summary": {
        "total": 0,
        "completed": 0,
        "percent_completed": 0
      },
      "body": "### Self Checks\n\n- [x] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).\n- [x] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.\n- [x] I confirm that I am using English to submit this report (ÊàëÂ∑≤ÈòÖËØªÂπ∂ÂêåÊÑè [Language Policy](https://github.com/langgenius/dify/issues/1542)).\n- [x] [FOR CHINESE USERS] ËØ∑Âä°ÂøÖ‰ΩøÁî®Ëã±ÊñáÊèê‰∫§ IssueÔºåÂê¶Âàô‰ºöË¢´ÂÖ≥Èó≠„ÄÇË∞¢Ë∞¢ÔºÅ:)\n- [x] Please do not modify this template :) and fill in all the required fields.\n\n### Dify version\n\n1.4.1\n\n### Cloud or Self Hosted\n\nSelf Hosted (Docker)\n\n### Steps to reproduce\n\n![Image](https://github.com/user-attachments/assets/4fe004f5-764c-4053-8156-8a0b19db7c27)\n![Image](https://github.com/user-attachments/assets/3d1d6c50-b4c1-47d6-a69a-9e7b6c6cbb1b)\n![Image](https://github.com/user-attachments/assets/9376cf93-3ea8-4eb6-99e7-9e14a826913b)\n\ni use the comfyui1.2.0, with the workflow (image to image), while public to tool, but there is no  place to get input image.\n\n### ‚úîÔ∏è Expected Behavior\n\n_No response_\n\n### ‚ùå Actual Behavior\n\n_No response_",
      "closed_by": {
        "login": "Jonathon-jin",
        "id": 174238636,
        "node_id": "U_kgDOCmKrrA",
        "avatar_url": "https://avatars.githubusercontent.com/u/174238636?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/Jonathon-jin",
        "html_url": "https://github.com/Jonathon-jin",
        "followers_url": "https://api.github.com/users/Jonathon-jin/followers",
        "following_url": "https://api.github.com/users/Jonathon-jin/following{/other_user}",
        "gists_url": "https://api.github.com/users/Jonathon-jin/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/Jonathon-jin/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/Jonathon-jin/subscriptions",
        "organizations_url": "https://api.github.com/users/Jonathon-jin/orgs",
        "repos_url": "https://api.github.com/users/Jonathon-jin/repos",
        "events_url": "https://api.github.com/users/Jonathon-jin/events{/privacy}",
        "received_events_url": "https://api.github.com/users/Jonathon-jin/received_events",
        "type": "User",
        "user_view_type": "public",
        "site_admin": false
      },
      "reactions": {
        "url": "https://api.github.com/repos/langgenius/dify/issues/20911/reactions",
        "total_count": 1,
        "+1": 1,
        "-1": 0,
        "laugh": 0,
        "hooray": 0,
        "confused": 0,
        "heart": 0,
        "rocket": 0,
        "eyes": 0
      },
      "timeline_url": "https://api.github.com/repos/langgenius/dify/issues/20911/timeline",
      "performed_via_github_app": null,
      "state_reason": "completed"
    },
    {
      "url": "https://api.github.com/repos/langgenius/dify/issues/20893",
      "repository_url": "https://api.github.com/repos/langgenius/dify",
      "labels_url": "https://api.github.com/repos/langgenius/dify/issues/20893/labels{/name}",
      "comments_url": "https://api.github.com/repos/langgenius/dify/issues/20893/comments",
      "events_url": "https://api.github.com/repos/langgenius/dify/issues/20893/events",
      "html_url": "https://github.com/langgenius/dify/issues/20893",
      "id": 3135373405,
      "node_id": "I_kwDOJVxJus664gBd",
      "number": 20893,
      "title": "The database uses MySQL and reports an error",
      "user": {
        "login": "NieRonghua",
        "id": 40586915,
        "node_id": "MDQ6VXNlcjQwNTg2OTE1",
        "avatar_url": "https://avatars.githubusercontent.com/u/40586915?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/NieRonghua",
        "html_url": "https://github.com/NieRonghua",
        "followers_url": "https://api.github.com/users/NieRonghua/followers",
        "following_url": "https://api.github.com/users/NieRonghua/following{/other_user}",
        "gists_url": "https://api.github.com/users/NieRonghua/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/NieRonghua/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/NieRonghua/subscriptions",
        "organizations_url": "https://api.github.com/users/NieRonghua/orgs",
        "repos_url": "https://api.github.com/users/NieRonghua/repos",
        "events_url": "https://api.github.com/users/NieRonghua/events{/privacy}",
        "received_events_url": "https://api.github.com/users/NieRonghua/received_events",
        "type": "User",
        "user_view_type": "public",
        "site_admin": false
      },
      "labels": [],
      "state": "closed",
      "locked": false,
      "assignee": null,
      "assignees": [],
      "milestone": null,
      "comments": 1,
      "created_at": "2025-06-11T03:37:35Z",
      "updated_at": "2025-06-12T06:17:17Z",
      "closed_at": "2025-06-12T06:17:17Z",
      "author_association": "NONE",
      "type": null,
      "active_lock_reason": null,
      "sub_issues_summary": {
        "total": 0,
        "completed": 0,
        "percent_completed": 0
      },
      "body": "### Self Checks\n\n- [x] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).\n- [x] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.\n- [x] I confirm that I am using English to submit this report (ÊàëÂ∑≤ÈòÖËØªÂπ∂ÂêåÊÑè [Language Policy](https://github.com/langgenius/dify/issues/1542)).\n- [x] [FOR CHINESE USERS] ËØ∑Âä°ÂøÖ‰ΩøÁî®Ëã±ÊñáÊèê‰∫§ IssueÔºåÂê¶Âàô‰ºöË¢´ÂÖ≥Èó≠„ÄÇË∞¢Ë∞¢ÔºÅ:)\n- [x] Please do not modify this template :) and fill in all the required fields.\n\n### Dify version\n\n1.4.1\n\n### Cloud or Self Hosted\n\nSelf Hosted (Source)\n\n### Steps to reproduce\n\nI deployed version 1.4.1 in a private environment, and the database used was MySQL. The error was as followsÔºö\n\n<img width=\"915\" alt=\"Image\" src=\"https://github.com/user-attachments/assets/9073492a-e670-4eaa-944d-112dca7c94be\" />\n\nThe print log shows that the value parameter type is str\n\n<img width=\"860\" alt=\"Image\" src=\"https://github.com/user-attachments/assets/001d247e-0fd9-4c79-accc-b402a50f7163\" />\n\nMy database config is defined as followsÔºö\n\n<img width=\"993\" alt=\"Image\" src=\"https://github.com/user-attachments/assets/2a5115dd-fcb6-4f3a-9542-20a76516dcc9\" />\n\n### ‚úîÔ∏è Expected Behavior\n\nI'm not sure if it's a configuration issue or a bug, hope to get an answer\n\n### ‚ùå Actual Behavior\n\n_No response_",
      "closed_by": {
        "login": "crazywoola",
        "id": 100913391,
        "node_id": "U_kgDOBgPQ7w",
        "avatar_url": "https://avatars.githubusercontent.com/u/100913391?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/crazywoola",
        "html_url": "https://github.com/crazywoola",
        "followers_url": "https://api.github.com/users/crazywoola/followers",
        "following_url": "https://api.github.com/users/crazywoola/following{/other_user}",
        "gists_url": "https://api.github.com/users/crazywoola/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/crazywoola/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/crazywoola/subscriptions",
        "organizations_url": "https://api.github.com/users/crazywoola/orgs",
        "repos_url": "https://api.github.com/users/crazywoola/repos",
        "events_url": "https://api.github.com/users/crazywoola/events{/privacy}",
        "received_events_url": "https://api.github.com/users/crazywoola/received_events",
        "type": "User",
        "user_view_type": "public",
        "site_admin": false
      },
      "reactions": {
        "url": "https://api.github.com/repos/langgenius/dify/issues/20893/reactions",
        "total_count": 1,
        "+1": 1,
        "-1": 0,
        "laugh": 0,
        "hooray": 0,
        "confused": 0,
        "heart": 0,
        "rocket": 0,
        "eyes": 0
      },
      "timeline_url": "https://api.github.com/repos/langgenius/dify/issues/20893/timeline",
      "performed_via_github_app": null,
      "state_reason": "not_planned"
    },
    {
      "url": "https://api.github.com/repos/langgenius/dify/issues/20887",
      "repository_url": "https://api.github.com/repos/langgenius/dify",
      "labels_url": "https://api.github.com/repos/langgenius/dify/issues/20887/labels{/name}",
      "comments_url": "https://api.github.com/repos/langgenius/dify/issues/20887/comments",
      "events_url": "https://api.github.com/repos/langgenius/dify/issues/20887/events",
      "html_url": "https://github.com/langgenius/dify/issues/20887",
      "id": 3135346249,
      "node_id": "I_kwDOJVxJus664ZZJ",
      "number": 20887,
      "title": "When Dify connects to the local Ollama model, the following errors often occur. May I ask what the reasons are and how to solve them? Thank you!",
      "user": {
        "login": "jiangxinufo",
        "id": 53360080,
        "node_id": "MDQ6VXNlcjUzMzYwMDgw",
        "avatar_url": "https://avatars.githubusercontent.com/u/53360080?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/jiangxinufo",
        "html_url": "https://github.com/jiangxinufo",
        "followers_url": "https://api.github.com/users/jiangxinufo/followers",
        "following_url": "https://api.github.com/users/jiangxinufo/following{/other_user}",
        "gists_url": "https://api.github.com/users/jiangxinufo/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/jiangxinufo/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/jiangxinufo/subscriptions",
        "organizations_url": "https://api.github.com/users/jiangxinufo/orgs",
        "repos_url": "https://api.github.com/users/jiangxinufo/repos",
        "events_url": "https://api.github.com/users/jiangxinufo/events{/privacy}",
        "received_events_url": "https://api.github.com/users/jiangxinufo/received_events",
        "type": "User",
        "user_view_type": "public",
        "site_admin": false
      },
      "labels": [
        {
          "id": 6210615164,
          "node_id": "LA_kwDOJVxJus8AAAABci53fA",
          "url": "https://api.github.com/repos/langgenius/dify/labels/%F0%9F%A4%94%20cant-reproduce",
          "name": "ü§î cant-reproduce",
          "color": "BDCE78",
          "default": false,
          "description": "We can not reproduce it or the information is limited"
        }
      ],
      "state": "closed",
      "locked": false,
      "assignee": null,
      "assignees": [],
      "milestone": null,
      "comments": 2,
      "created_at": "2025-06-11T03:14:23Z",
      "updated_at": "2025-06-12T06:14:59Z",
      "closed_at": "2025-06-12T06:14:38Z",
      "author_association": "NONE",
      "type": null,
      "active_lock_reason": null,
      "sub_issues_summary": {
        "total": 0,
        "completed": 0,
        "percent_completed": 0
      },
      "body": "### Self Checks\n\n- [x] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).\n- [x] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.\n- [x] I confirm that I am using English to submit this report (ÊàëÂ∑≤ÈòÖËØªÂπ∂ÂêåÊÑè [Language Policy](https://github.com/langgenius/dify/issues/1542)).\n- [x] [FOR CHINESE USERS] ËØ∑Âä°ÂøÖ‰ΩøÁî®Ëã±ÊñáÊèê‰∫§ IssueÔºåÂê¶Âàô‰ºöË¢´ÂÖ≥Èó≠„ÄÇË∞¢Ë∞¢ÔºÅ:)\n- [x] Please do not modify this template :) and fill in all the required fields.\n\n### Dify version\n\n1.2.0\n\n### Cloud or Self Hosted\n\nSelf Hosted (Docker)\n\n### Steps to reproduce\n\nWhen Dify connects to the local Ollama model, the following errors often occur. May I ask what the reasons are and how to solve them? Thank you!\n\nRun failed: [models] Server Unavailable Error, HTTPConnectionPool(host='172.18.100.163', port=8888): Max retries exceeded with url: /api/chat (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f55438eaa50>, 'Connection to 172.18.100.163 timed out. (connect timeout=10)'))\n\n### ‚úîÔ∏è Expected Behavior\n\nRun failed: [models] Server Unavailable Error, HTTPConnectionPool(host='172.18.100.163', port=8888): Max retries exceeded with url: /api/chat (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f55438eaa50>, 'Connection to 172.18.100.163 timed out. (connect timeout=10)'))\n\n### ‚ùå Actual Behavior\n\n_No response_",
      "closed_by": {
        "login": "crazywoola",
        "id": 100913391,
        "node_id": "U_kgDOBgPQ7w",
        "avatar_url": "https://avatars.githubusercontent.com/u/100913391?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/crazywoola",
        "html_url": "https://github.com/crazywoola",
        "followers_url": "https://api.github.com/users/crazywoola/followers",
        "following_url": "https://api.github.com/users/crazywoola/following{/other_user}",
        "gists_url": "https://api.github.com/users/crazywoola/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/crazywoola/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/crazywoola/subscriptions",
        "organizations_url": "https://api.github.com/users/crazywoola/orgs",
        "repos_url": "https://api.github.com/users/crazywoola/repos",
        "events_url": "https://api.github.com/users/crazywoola/events{/privacy}",
        "received_events_url": "https://api.github.com/users/crazywoola/received_events",
        "type": "User",
        "user_view_type": "public",
        "site_admin": false
      },
      "reactions": {
        "url": "https://api.github.com/repos/langgenius/dify/issues/20887/reactions",
        "total_count": 1,
        "+1": 1,
        "-1": 0,
        "laugh": 0,
        "hooray": 0,
        "confused": 0,
        "heart": 0,
        "rocket": 0,
        "eyes": 0
      },
      "timeline_url": "https://api.github.com/repos/langgenius/dify/issues/20887/timeline",
      "performed_via_github_app": null,
      "state_reason": "duplicate"
    },
    {
      "url": "https://api.github.com/repos/langgenius/dify/issues/20884",
      "repository_url": "https://api.github.com/repos/langgenius/dify",
      "labels_url": "https://api.github.com/repos/langgenius/dify/issues/20884/labels{/name}",
      "comments_url": "https://api.github.com/repos/langgenius/dify/issues/20884/comments",
      "events_url": "https://api.github.com/repos/langgenius/dify/issues/20884/events",
      "html_url": "https://github.com/langgenius/dify/issues/20884",
      "id": 3135295590,
      "node_id": "I_kwDOJVxJus664NBm",
      "number": 20884,
      "title": "Modify port 80 issue",
      "user": {
        "login": "AmShadows",
        "id": 37405406,
        "node_id": "MDQ6VXNlcjM3NDA1NDA2",
        "avatar_url": "https://avatars.githubusercontent.com/u/37405406?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/AmShadows",
        "html_url": "https://github.com/AmShadows",
        "followers_url": "https://api.github.com/users/AmShadows/followers",
        "following_url": "https://api.github.com/users/AmShadows/following{/other_user}",
        "gists_url": "https://api.github.com/users/AmShadows/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/AmShadows/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/AmShadows/subscriptions",
        "organizations_url": "https://api.github.com/users/AmShadows/orgs",
        "repos_url": "https://api.github.com/users/AmShadows/repos",
        "events_url": "https://api.github.com/users/AmShadows/events{/privacy}",
        "received_events_url": "https://api.github.com/users/AmShadows/received_events",
        "type": "User",
        "user_view_type": "public",
        "site_admin": false
      },
      "labels": [],
      "state": "closed",
      "locked": false,
      "assignee": null,
      "assignees": [],
      "milestone": null,
      "comments": 1,
      "created_at": "2025-06-11T02:38:18Z",
      "updated_at": "2025-06-12T06:15:08Z",
      "closed_at": "2025-06-12T06:15:08Z",
      "author_association": "NONE",
      "type": null,
      "active_lock_reason": null,
      "sub_issues_summary": {
        "total": 0,
        "completed": 0,
        "percent_completed": 0
      },
      "body": "### Self Checks\n\n- [x] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).\n- [x] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.\n- [x] I confirm that I am using English to submit this report (ÊàëÂ∑≤ÈòÖËØªÂπ∂ÂêåÊÑè [Language Policy](https://github.com/langgenius/dify/issues/1542)).\n- [x] [FOR CHINESE USERS] ËØ∑Âä°ÂøÖ‰ΩøÁî®Ëã±ÊñáÊèê‰∫§ IssueÔºåÂê¶Âàô‰ºöË¢´ÂÖ≥Èó≠„ÄÇË∞¢Ë∞¢ÔºÅ:)\n- [x] Please do not modify this template :) and fill in all the required fields.\n\n### Dify version\n\nv1.4.1\n\n### Cloud or Self Hosted\n\nSelf Hosted (Docker)\n\n### Steps to reproduce\n\nwebÂ∫îÁî®‰øÆÊîπ80‰∏∫8888Á´ØÂè£‰πãÂêéÔºåÂèëÂ∏ÉÊõ¥Êñ∞ÈáåÈù¢ÁöÑËøêË°å‰æùÁÑ∂ËøòÊòØ80Á´ØÂè£Ë∑≥ËΩ¨ËÆøÈóÆ„ÄÇ‰æãÂ¶Ç1.1.1.1:80/chat/shdusdjhklf\n\n![Image](https://github.com/user-attachments/assets/ff439ab0-4ecd-40f3-8fa4-639ff5d51729)\n\n### ‚úîÔ∏è Expected Behavior\n\n1.1.1.1:8888/chat/shdusdjhklf\n\n### ‚ùå Actual Behavior\n\n1.1.1.1:80/chat/shdusdjhklf",
      "closed_by": {
        "login": "crazywoola",
        "id": 100913391,
        "node_id": "U_kgDOBgPQ7w",
        "avatar_url": "https://avatars.githubusercontent.com/u/100913391?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/crazywoola",
        "html_url": "https://github.com/crazywoola",
        "followers_url": "https://api.github.com/users/crazywoola/followers",
        "following_url": "https://api.github.com/users/crazywoola/following{/other_user}",
        "gists_url": "https://api.github.com/users/crazywoola/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/crazywoola/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/crazywoola/subscriptions",
        "organizations_url": "https://api.github.com/users/crazywoola/orgs",
        "repos_url": "https://api.github.com/users/crazywoola/repos",
        "events_url": "https://api.github.com/users/crazywoola/events{/privacy}",
        "received_events_url": "https://api.github.com/users/crazywoola/received_events",
        "type": "User",
        "user_view_type": "public",
        "site_admin": false
      },
      "reactions": {
        "url": "https://api.github.com/repos/langgenius/dify/issues/20884/reactions",
        "total_count": 1,
        "+1": 1,
        "-1": 0,
        "laugh": 0,
        "hooray": 0,
        "confused": 0,
        "heart": 0,
        "rocket": 0,
        "eyes": 0
      },
      "timeline_url": "https://api.github.com/repos/langgenius/dify/issues/20884/timeline",
      "performed_via_github_app": null,
      "state_reason": "not_planned"
    },
    {
      "url": "https://api.github.com/repos/langgenius/dify/issues/20863",
      "repository_url": "https://api.github.com/repos/langgenius/dify",
      "labels_url": "https://api.github.com/repos/langgenius/dify/issues/20863/labels{/name}",
      "comments_url": "https://api.github.com/repos/langgenius/dify/issues/20863/comments",
      "events_url": "https://api.github.com/repos/langgenius/dify/issues/20863/events",
      "html_url": "https://github.com/langgenius/dify/issues/20863",
      "id": 3132291455,
      "node_id": "I_kwDOJVxJus66svl_",
      "number": 20863,
      "title": "Calling the chart message interface does not support streaming output, using the Volcano Ark DeepseekV3 model",
      "user": {
        "login": "htk949",
        "id": 8454387,
        "node_id": "MDQ6VXNlcjg0NTQzODc=",
        "avatar_url": "https://avatars.githubusercontent.com/u/8454387?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/htk949",
        "html_url": "https://github.com/htk949",
        "followers_url": "https://api.github.com/users/htk949/followers",
        "following_url": "https://api.github.com/users/htk949/following{/other_user}",
        "gists_url": "https://api.github.com/users/htk949/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/htk949/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/htk949/subscriptions",
        "organizations_url": "https://api.github.com/users/htk949/orgs",
        "repos_url": "https://api.github.com/users/htk949/repos",
        "events_url": "https://api.github.com/users/htk949/events{/privacy}",
        "received_events_url": "https://api.github.com/users/htk949/received_events",
        "type": "User",
        "user_view_type": "public",
        "site_admin": false
      },
      "labels": [],
      "state": "closed",
      "locked": false,
      "assignee": null,
      "assignees": [],
      "milestone": null,
      "comments": 2,
      "created_at": "2025-06-10T06:21:53Z",
      "updated_at": "2025-06-12T08:20:24Z",
      "closed_at": "2025-06-12T08:20:24Z",
      "author_association": "NONE",
      "type": null,
      "active_lock_reason": null,
      "sub_issues_summary": {
        "total": 0,
        "completed": 0,
        "percent_completed": 0
      },
      "body": "### Self Checks\n\n- [x] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).\n- [x] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.\n- [x] I confirm that I am using English to submit this report (ÊàëÂ∑≤ÈòÖËØªÂπ∂ÂêåÊÑè [Language Policy](https://github.com/langgenius/dify/issues/1542)).\n- [x] [FOR CHINESE USERS] ËØ∑Âä°ÂøÖ‰ΩøÁî®Ëã±ÊñáÊèê‰∫§ IssueÔºåÂê¶Âàô‰ºöË¢´ÂÖ≥Èó≠„ÄÇË∞¢Ë∞¢ÔºÅ:)\n- [x] Please do not modify this template :) and fill in all the required fields.\n\n### Dify version\n\n1.3.0\n\n### Cloud or Self Hosted\n\nSelf Hosted (Docker)\n\n### Steps to reproduce\n\n**Problem descriptionÔºö**\n1.Is the process node set in chatflow unable to stream output due to improper processing of a certain node?\n\n![Image](https://github.com/user-attachments/assets/ce03c631-6fd6-433e-9f38-5d9ef6beaec1)\n\n2.The call to the chat message interface's responsive_mode parameter passed streaming, but the streaming output did not work and returned blocked output\n\n![Image](https://github.com/user-attachments/assets/6c9a2b20-c91a-4932-a2e4-66e2631a73a9)\n\n![Image](https://github.com/user-attachments/assets/e6138d53-82ef-4cb4-a623-ce8376dbca70)\n\n### ‚úîÔ∏è Expected Behavior\n\n_No response_\n\n### ‚ùå Actual Behavior\n\n_No response_",
      "closed_by": {
        "login": "htk949",
        "id": 8454387,
        "node_id": "MDQ6VXNlcjg0NTQzODc=",
        "avatar_url": "https://avatars.githubusercontent.com/u/8454387?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/htk949",
        "html_url": "https://github.com/htk949",
        "followers_url": "https://api.github.com/users/htk949/followers",
        "following_url": "https://api.github.com/users/htk949/following{/other_user}",
        "gists_url": "https://api.github.com/users/htk949/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/htk949/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/htk949/subscriptions",
        "organizations_url": "https://api.github.com/users/htk949/orgs",
        "repos_url": "https://api.github.com/users/htk949/repos",
        "events_url": "https://api.github.com/users/htk949/events{/privacy}",
        "received_events_url": "https://api.github.com/users/htk949/received_events",
        "type": "User",
        "user_view_type": "public",
        "site_admin": false
      },
      "reactions": {
        "url": "https://api.github.com/repos/langgenius/dify/issues/20863/reactions",
        "total_count": 1,
        "+1": 1,
        "-1": 0,
        "laugh": 0,
        "hooray": 0,
        "confused": 0,
        "heart": 0,
        "rocket": 0,
        "eyes": 0
      },
      "timeline_url": "https://api.github.com/repos/langgenius/dify/issues/20863/timeline",
      "performed_via_github_app": null,
      "state_reason": "completed"
    },
    {
      "url": "https://api.github.com/repos/langgenius/dify/issues/20856",
      "repository_url": "https://api.github.com/repos/langgenius/dify",
      "labels_url": "https://api.github.com/repos/langgenius/dify/issues/20856/labels{/name}",
      "comments_url": "https://api.github.com/repos/langgenius/dify/issues/20856/comments",
      "events_url": "https://api.github.com/repos/langgenius/dify/issues/20856/events",
      "html_url": "https://github.com/langgenius/dify/issues/20856",
      "id": 3132061331,
      "node_id": "I_kwDOJVxJus66r3aT",
      "number": 20856,
      "title": "An error occurred during credentials validation: HTTPConnectionPool(host='ollama', port=11434): Failed to resolve 'ollama' ([Errno -3] Temporary failure in name resolution)\"))",
      "user": {
        "login": "Ikaros-521",
        "id": 40910637,
        "node_id": "MDQ6VXNlcjQwOTEwNjM3",
        "avatar_url": "https://avatars.githubusercontent.com/u/40910637?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/Ikaros-521",
        "html_url": "https://github.com/Ikaros-521",
        "followers_url": "https://api.github.com/users/Ikaros-521/followers",
        "following_url": "https://api.github.com/users/Ikaros-521/following{/other_user}",
        "gists_url": "https://api.github.com/users/Ikaros-521/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/Ikaros-521/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/Ikaros-521/subscriptions",
        "organizations_url": "https://api.github.com/users/Ikaros-521/orgs",
        "repos_url": "https://api.github.com/users/Ikaros-521/repos",
        "events_url": "https://api.github.com/users/Ikaros-521/events{/privacy}",
        "received_events_url": "https://api.github.com/users/Ikaros-521/received_events",
        "type": "User",
        "user_view_type": "public",
        "site_admin": false
      },
      "labels": [
        {
          "id": 5379590677,
          "node_id": "LA_kwDOJVxJus8AAAABQKYKFQ",
          "url": "https://api.github.com/repos/langgenius/dify/labels/%F0%9F%90%9E%20bug",
          "name": "üêû bug",
          "color": "FFC300",
          "default": false,
          "description": "Something isn't working"
        },
        {
          "id": 7777942092,
          "node_id": "LA_kwDOJVxJus8AAAABz5n6TA",
          "url": "https://api.github.com/repos/langgenius/dify/labels/cloud",
          "name": "cloud",
          "color": "E0091D",
          "default": false,
          "description": "When the version is cloud and it is a bug report"
        }
      ],
      "state": "closed",
      "locked": false,
      "assignee": null,
      "assignees": [],
      "milestone": null,
      "comments": 6,
      "created_at": "2025-06-10T03:57:20Z",
      "updated_at": "2025-06-12T07:37:20Z",
      "closed_at": "2025-06-10T06:14:27Z",
      "author_association": "NONE",
      "type": null,
      "active_lock_reason": null,
      "sub_issues_summary": {
        "total": 0,
        "completed": 0,
        "percent_completed": 0
      },
      "body": "### Self Checks\n\n- [x] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).\n- [x] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.\n- [x] I confirm that I am using English to submit this report (ÊàëÂ∑≤ÈòÖËØªÂπ∂ÂêåÊÑè [Language Policy](https://github.com/langgenius/dify/issues/1542)).\n- [x] [FOR CHINESE USERS] ËØ∑Âä°ÂøÖ‰ΩøÁî®Ëã±ÊñáÊèê‰∫§ IssueÔºåÂê¶Âàô‰ºöË¢´ÂÖ≥Èó≠„ÄÇË∞¢Ë∞¢ÔºÅ:)\n- [x] Please do not modify this template :) and fill in all the required fields.\n\n### Dify version\n\n1.1.3\n\n### Cloud or Self Hosted\n\nCloud\n\n### Steps to reproduce\n\n![Image](https://github.com/user-attachments/assets/88201bef-2f24-4a91-878f-f0e9e1d5f264)\n\n```\nAn error occurred during credentials validation: HTTPConnectionPool(host='ollama', port=11434): Max retries exceeded with url: /api/chat (Caused by NameResolutionError(\"<urllib3.connection.HTTPConnection object at 0x72cdcc922c60>: Failed to resolve 'ollama' ([Errno -3] Temporary failure in name resolution)\"))\n```\n\nyaml\n![Image](https://github.com/user-attachments/assets/56db4645-d7da-4350-bd65-564aa23f85e4)\n\n![Image](https://github.com/user-attachments/assets/e87047e7-7c6a-4591-af3f-b1f1b5fb6d05)\n\nin docker_api_1, I can use `curl http://ollama:11434` to connect ollama serverÔºåbut in webui is error\n\n### ‚úîÔ∏è Expected Behavior\n\nnormal use\n\n### ‚ùå Actual Behavior\n\n_No response_",
      "closed_by": {
        "login": "crazywoola",
        "id": 100913391,
        "node_id": "U_kgDOBgPQ7w",
        "avatar_url": "https://avatars.githubusercontent.com/u/100913391?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/crazywoola",
        "html_url": "https://github.com/crazywoola",
        "followers_url": "https://api.github.com/users/crazywoola/followers",
        "following_url": "https://api.github.com/users/crazywoola/following{/other_user}",
        "gists_url": "https://api.github.com/users/crazywoola/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/crazywoola/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/crazywoola/subscriptions",
        "organizations_url": "https://api.github.com/users/crazywoola/orgs",
        "repos_url": "https://api.github.com/users/crazywoola/repos",
        "events_url": "https://api.github.com/users/crazywoola/events{/privacy}",
        "received_events_url": "https://api.github.com/users/crazywoola/received_events",
        "type": "User",
        "user_view_type": "public",
        "site_admin": false
      },
      "reactions": {
        "url": "https://api.github.com/repos/langgenius/dify/issues/20856/reactions",
        "total_count": 1,
        "+1": 1,
        "-1": 0,
        "laugh": 0,
        "hooray": 0,
        "confused": 0,
        "heart": 0,
        "rocket": 0,
        "eyes": 0
      },
      "timeline_url": "https://api.github.com/repos/langgenius/dify/issues/20856/timeline",
      "performed_via_github_app": null,
      "state_reason": "not_planned"
    },
    {
      "url": "https://api.github.com/repos/langgenius/dify/issues/20779",
      "repository_url": "https://api.github.com/repos/langgenius/dify",
      "labels_url": "https://api.github.com/repos/langgenius/dify/issues/20779/labels{/name}",
      "comments_url": "https://api.github.com/repos/langgenius/dify/issues/20779/comments",
      "events_url": "https://api.github.com/repos/langgenius/dify/issues/20779/events",
      "html_url": "https://github.com/langgenius/dify/issues/20779",
      "id": 3126822079,
      "node_id": "I_kwDOJVxJus66X4S_",
      "number": 20779,
      "title": "Langfuse Integration: Internal Nodes of Dify Workflows Are Not Being Traced",
      "user": {
        "login": "huangyafei",
        "id": 1579347,
        "node_id": "MDQ6VXNlcjE1NzkzNDc=",
        "avatar_url": "https://avatars.githubusercontent.com/u/1579347?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/huangyafei",
        "html_url": "https://github.com/huangyafei",
        "followers_url": "https://api.github.com/users/huangyafei/followers",
        "following_url": "https://api.github.com/users/huangyafei/following{/other_user}",
        "gists_url": "https://api.github.com/users/huangyafei/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/huangyafei/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/huangyafei/subscriptions",
        "organizations_url": "https://api.github.com/users/huangyafei/orgs",
        "repos_url": "https://api.github.com/users/huangyafei/repos",
        "events_url": "https://api.github.com/users/huangyafei/events{/privacy}",
        "received_events_url": "https://api.github.com/users/huangyafei/received_events",
        "type": "User",
        "user_view_type": "public",
        "site_admin": false
      },
      "labels": [
        {
          "id": 6736080616,
          "node_id": "LA_kwDOJVxJus8AAAABkYBu6A",
          "url": "https://api.github.com/repos/langgenius/dify/labels/%F0%9F%8C%8A%20feat:workflow",
          "name": "üåä feat:workflow",
          "color": "D1CF8C",
          "default": false,
          "description": "Workflow related stuff."
        }
      ],
      "state": "closed",
      "locked": false,
      "assignee": null,
      "assignees": [],
      "milestone": null,
      "comments": 9,
      "created_at": "2025-06-07T10:17:38Z",
      "updated_at": "2025-06-12T10:01:21Z",
      "closed_at": "2025-06-12T10:01:21Z",
      "author_association": "CONTRIBUTOR",
      "type": null,
      "active_lock_reason": null,
      "sub_issues_summary": {
        "total": 0,
        "completed": 0,
        "percent_completed": 0
      },
      "body": "### Self Checks\n\n- [x] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).\n- [x] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.\n- [x] I confirm that I am using English to submit this report (ÊàëÂ∑≤ÈòÖËØªÂπ∂ÂêåÊÑè [Language Policy](https://github.com/langgenius/dify/issues/1542)).\n- [x] [FOR CHINESE USERS] ËØ∑Âä°ÂøÖ‰ΩøÁî®Ëã±ÊñáÊèê‰∫§ IssueÔºåÂê¶Âàô‰ºöË¢´ÂÖ≥Èó≠„ÄÇË∞¢Ë∞¢ÔºÅ:)\n- [x] Please do not modify this template :) and fill in all the required fields.\n\n### Dify version\n\n1.4.1\n\n### Cloud or Self Hosted\n\nSelf Hosted (Docker)\n\n### Steps to reproduce\n\nHi team,\n\nI've encountered an issue while using the Langfuse integration to monitor my Dify workflows.\n\nWhile Langfuse correctly captures the overall workflow's inputs, outputs, and metadata, it fails to trace the detailed execution of the individual nodes within the workflow.\n\nThis can be observed in the attached screenshots with a simple example:\n\nThe sample workflow is straightforward: It takes \"foo\" as an input, processes it through a Code node, and outputs \"bar\".\n\n![Image](https://github.com/user-attachments/assets/04c83731-0a39-4c3d-974b-705460de5dae)\n\nIn the Dify UI: The native tracing panel on the right correctly displays the detailed execution trace, showing each node's activity as expected.\n\n![Image](https://github.com/user-attachments/assets/ec32aea4-33c6-4f71-a493-76530f9cd962)\n\nIn the Langfuse UI: However, only the top-level trace for the entire workflow (with its final input and output) is visible. The details of the internal nodes, such as the Code node, are completely missing.\n\n![Image](https://github.com/user-attachments/assets/5a08ac90-a1aa-42e8-ab7f-a3aca6b77d77)\n\nI used this Langfuse integration frequently last year to trace my workflows, and everything worked perfectly back then. I haven't used this feature for a while, so I'm not sure why it's no longer tracking the internal node details.\n\nThank you for looking into this!\n\n### ‚úîÔ∏è Expected Behavior\n\nThis makes it difficult to debug or analyze the performance of individual steps within a complex workflow using Langfuse. It would be great if the trace could include the full execution path, just like it appears in Dify's own tracer.\n\n### ‚ùå Actual Behavior\n\n_No response_",
      "closed_by": {
        "login": "huangyafei",
        "id": 1579347,
        "node_id": "MDQ6VXNlcjE1NzkzNDc=",
        "avatar_url": "https://avatars.githubusercontent.com/u/1579347?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/huangyafei",
        "html_url": "https://github.com/huangyafei",
        "followers_url": "https://api.github.com/users/huangyafei/followers",
        "following_url": "https://api.github.com/users/huangyafei/following{/other_user}",
        "gists_url": "https://api.github.com/users/huangyafei/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/huangyafei/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/huangyafei/subscriptions",
        "organizations_url": "https://api.github.com/users/huangyafei/orgs",
        "repos_url": "https://api.github.com/users/huangyafei/repos",
        "events_url": "https://api.github.com/users/huangyafei/events{/privacy}",
        "received_events_url": "https://api.github.com/users/huangyafei/received_events",
        "type": "User",
        "user_view_type": "public",
        "site_admin": false
      },
      "reactions": {
        "url": "https://api.github.com/repos/langgenius/dify/issues/20779/reactions",
        "total_count": 3,
        "+1": 3,
        "-1": 0,
        "laugh": 0,
        "hooray": 0,
        "confused": 0,
        "heart": 0,
        "rocket": 0,
        "eyes": 0
      },
      "timeline_url": "https://api.github.com/repos/langgenius/dify/issues/20779/timeline",
      "performed_via_github_app": null,
      "state_reason": "completed"
    },
    {
      "url": "https://api.github.com/repos/langgenius/dify/issues/14848",
      "repository_url": "https://api.github.com/repos/langgenius/dify",
      "labels_url": "https://api.github.com/repos/langgenius/dify/issues/14848/labels{/name}",
      "comments_url": "https://api.github.com/repos/langgenius/dify/issues/14848/comments",
      "events_url": "https://api.github.com/repos/langgenius/dify/issues/14848/events",
      "html_url": "https://github.com/langgenius/dify/issues/14848",
      "id": 2892941827,
      "node_id": "I_kwDOJVxJus6sbsoD",
      "number": 14848,
      "title": "ÂÆåÂÖ®Á¶ªÁ∫øÈÉ®ÁΩ≤difyÊó∂ÔºåÊó†Ê≥ïÂÆâË£ÖÊèí‰ª∂",
      "user": {
        "login": "unclelaotie",
        "id": 199236721,
        "node_id": "U_kgDOC-AccQ",
        "avatar_url": "https://avatars.githubusercontent.com/u/199236721?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/unclelaotie",
        "html_url": "https://github.com/unclelaotie",
        "followers_url": "https://api.github.com/users/unclelaotie/followers",
        "following_url": "https://api.github.com/users/unclelaotie/following{/other_user}",
        "gists_url": "https://api.github.com/users/unclelaotie/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/unclelaotie/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/unclelaotie/subscriptions",
        "organizations_url": "https://api.github.com/users/unclelaotie/orgs",
        "repos_url": "https://api.github.com/users/unclelaotie/repos",
        "events_url": "https://api.github.com/users/unclelaotie/events{/privacy}",
        "received_events_url": "https://api.github.com/users/unclelaotie/received_events",
        "type": "User",
        "user_view_type": "public",
        "site_admin": false
      },
      "labels": [
        {
          "id": 5379590677,
          "node_id": "LA_kwDOJVxJus8AAAABQKYKFQ",
          "url": "https://api.github.com/repos/langgenius/dify/labels/%F0%9F%90%9E%20bug",
          "name": "üêû bug",
          "color": "FFC300",
          "default": false,
          "description": "Something isn't working"
        }
      ],
      "state": "closed",
      "locked": false,
      "assignee": null,
      "assignees": [],
      "milestone": null,
      "comments": 27,
      "created_at": "2025-03-04T05:00:56Z",
      "updated_at": "2025-06-12T07:30:35Z",
      "closed_at": "2025-03-04T06:09:32Z",
      "author_association": "NONE",
      "type": null,
      "active_lock_reason": null,
      "sub_issues_summary": {
        "total": 0,
        "completed": 0,
        "percent_completed": 0
      },
      "body": "### Self Checks\n\n- [x] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).\n- [x] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.\n- [x] I confirm that I am using English to submit this report (ÊàëÂ∑≤ÈòÖËØªÂπ∂ÂêåÊÑè [Language Policy](https://github.com/langgenius/dify/issues/1542)).\n- [x] [FOR CHINESE USERS] ËØ∑Âä°ÂøÖ‰ΩøÁî®Ëã±ÊñáÊèê‰∫§ IssueÔºåÂê¶Âàô‰ºöË¢´ÂÖ≥Èó≠„ÄÇË∞¢Ë∞¢ÔºÅ:Ôºâ\n- [x] Please do not modify this template :) and fill in all the required fields.\n\n### Dify version\n\n1.0.0\n\n### Cloud or Self Hosted\n\nSelf Hosted (Docker)\n\n### Steps to reproduce\n\nÂ∞ÜdifyÊâÄÊúâdockerÂåÖÂíå‰ªéÊèí‰ª∂Â∏ÇÂú∫‰∏ãËΩΩÁöÑÊèí‰ª∂ÂØºÂÖ•Á¶ªÁ∫øÁéØÂ¢ÉÂêéÔºåÈÉ®ÁΩ≤difyÂÆåÊàêÂπ∂ÂêØÂä®ÂêéÔºå‰ªéÊú¨Âú∞ÂÆâË£ÖÊèí‰ª∂Â§±Ë¥•ÔºåÊü•ÁúãÊó•ÂøóÊòæÁ§∫‰∏ãËΩΩdify_pluginÁöÑÁõ∏ÂÖ≥pythonÂåÖÂ§±Ë¥•„ÄÇÂ∏åÊúõÂèØ‰ª•Â∞ÜÊØè‰∏™Êèí‰ª∂Áõ∏ÂÖ≥ÁöÑdify_pluginÂèäÂÖ∂‰ªñpythonÂåÖ‰∏éÊèí‰ª∂ÊâìÂåÖÂú®‰∏ÄËµ∑ÔºåÊñπ‰æøÁ¶ªÁ∫øÁî®Êà∑ÂÆâË£Ö‰ΩøÁî®„ÄÇ\n\n### ‚úîÔ∏è Expected Behavior\n\nÁ¶ªÁ∫øÁéØÂ¢ÉÂèØ‰ª•Ê≠£Â∏∏ÂÆâË£ÖÂêÑÁßçÊèí‰ª∂„ÄÇ\n\n### ‚ùå Actual Behavior\n\nÁ¶ªÁ∫øÁéØÂ¢ÉÊó†Ê≥ïÂÆâË£ÖÊèí‰ª∂„ÄÇ",
      "closed_by": {
        "login": "crazywoola",
        "id": 100913391,
        "node_id": "U_kgDOBgPQ7w",
        "avatar_url": "https://avatars.githubusercontent.com/u/100913391?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/crazywoola",
        "html_url": "https://github.com/crazywoola",
        "followers_url": "https://api.github.com/users/crazywoola/followers",
        "following_url": "https://api.github.com/users/crazywoola/following{/other_user}",
        "gists_url": "https://api.github.com/users/crazywoola/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/crazywoola/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/crazywoola/subscriptions",
        "organizations_url": "https://api.github.com/users/crazywoola/orgs",
        "repos_url": "https://api.github.com/users/crazywoola/repos",
        "events_url": "https://api.github.com/users/crazywoola/events{/privacy}",
        "received_events_url": "https://api.github.com/users/crazywoola/received_events",
        "type": "User",
        "user_view_type": "public",
        "site_admin": false
      },
      "reactions": {
        "url": "https://api.github.com/repos/langgenius/dify/issues/14848/reactions",
        "total_count": 8,
        "+1": 8,
        "-1": 0,
        "laugh": 0,
        "hooray": 0,
        "confused": 0,
        "heart": 0,
        "rocket": 0,
        "eyes": 0
      },
      "timeline_url": "https://api.github.com/repos/langgenius/dify/issues/14848/timeline",
      "performed_via_github_app": null,
      "state_reason": "completed"
    },
    {
      "url": "https://api.github.com/repos/langgenius/dify/issues/14199",
      "repository_url": "https://api.github.com/repos/langgenius/dify",
      "labels_url": "https://api.github.com/repos/langgenius/dify/issues/14199/labels{/name}",
      "comments_url": "https://api.github.com/repos/langgenius/dify/issues/14199/comments",
      "events_url": "https://api.github.com/repos/langgenius/dify/issues/14199/events",
      "html_url": "https://github.com/langgenius/dify/issues/14199",
      "id": 2870734487,
      "node_id": "I_kwDOJVxJus6rG-6X",
      "number": 14199,
      "title": "An error occurred during database initialization",
      "user": {
        "login": "lovezhou1990",
        "id": 2314874,
        "node_id": "MDQ6VXNlcjIzMTQ4NzQ=",
        "avatar_url": "https://avatars.githubusercontent.com/u/2314874?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/lovezhou1990",
        "html_url": "https://github.com/lovezhou1990",
        "followers_url": "https://api.github.com/users/lovezhou1990/followers",
        "following_url": "https://api.github.com/users/lovezhou1990/following{/other_user}",
        "gists_url": "https://api.github.com/users/lovezhou1990/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/lovezhou1990/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/lovezhou1990/subscriptions",
        "organizations_url": "https://api.github.com/users/lovezhou1990/orgs",
        "repos_url": "https://api.github.com/users/lovezhou1990/repos",
        "events_url": "https://api.github.com/users/lovezhou1990/events{/privacy}",
        "received_events_url": "https://api.github.com/users/lovezhou1990/received_events",
        "type": "User",
        "user_view_type": "public",
        "site_admin": false
      },
      "labels": [
        {
          "id": 6210615164,
          "node_id": "LA_kwDOJVxJus8AAAABci53fA",
          "url": "https://api.github.com/repos/langgenius/dify/labels/%F0%9F%A4%94%20cant-reproduce",
          "name": "ü§î cant-reproduce",
          "color": "BDCE78",
          "default": false,
          "description": "We can not reproduce it or the information is limited"
        }
      ],
      "state": "closed",
      "locked": false,
      "assignee": null,
      "assignees": [],
      "milestone": null,
      "comments": 5,
      "created_at": "2025-02-22T13:15:52Z",
      "updated_at": "2025-06-12T03:53:39Z",
      "closed_at": "2025-02-23T05:21:34Z",
      "author_association": "NONE",
      "type": null,
      "active_lock_reason": null,
      "sub_issues_summary": {
        "total": 0,
        "completed": 0,
        "percent_completed": 0
      },
      "body": "### Self Checks\n\n- [x] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).\n- [x] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.\n- [x] I confirm that I am using English to submit this report (ÊàëÂ∑≤ÈòÖËØªÂπ∂ÂêåÊÑè [Language Policy](https://github.com/langgenius/dify/issues/1542)).\n- [x] [FOR CHINESE USERS] ËØ∑Âä°ÂøÖ‰ΩøÁî®Ëã±ÊñáÊèê‰∫§ IssueÔºåÂê¶Âàô‰ºöË¢´ÂÖ≥Èó≠„ÄÇË∞¢Ë∞¢ÔºÅ:Ôºâ\n- [x] Please do not modify this template :) and fill in all the required fields.\n\n### Dify version\n\nLatest \n\n### Cloud or Self Hosted\n\nSelf Hosted (Source)\n\n### Steps to reproduce\n\n```\nAfter git clone the latest code, run the \"flask db upgrade\" command according to the installation procedure and the following error message is displayed:\n(non-package-mode-py3.12) D:\\workspace\\opensource\\dify_ws\\dify\\api>flask db upgrade\nTraceback (most recent call last):\n  File \"<frozen runpy>\", line 198, in _run_module_as_main\n  File \"<frozen runpy>\", line 88, in _run_code\n  File \"D:\\workspace\\opensource\\dify_ws\\dify\\api\\.venv\\Scripts\\flask.exe\\__main__.py\", line 7, in <module>\n  File \"D:\\workspace\\opensource\\dify_ws\\dify\\api\\.venv\\Lib\\site-packages\\flask\\cli.py\", line 1129, in main\n    cli.main()\n  File \"D:\\workspace\\opensource\\dify_ws\\dify\\api\\.venv\\Lib\\site-packages\\click\\core.py\", line 1082, in main\n    rv = self.invoke(ctx)\n         ^^^^^^^^^^^^^^^^\n  File \"D:\\workspace\\opensource\\dify_ws\\dify\\api\\.venv\\Lib\\site-packages\\click\\core.py\", line 1697, in invoke\n    return _process_result(sub_ctx.command.invoke(sub_ctx))\n                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\workspace\\opensource\\dify_ws\\dify\\api\\.venv\\Lib\\site-packages\\click\\core.py\", line 1697, in invoke\n    return _process_result(sub_ctx.command.invoke(sub_ctx))\n                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\workspace\\opensource\\dify_ws\\dify\\api\\.venv\\Lib\\site-packages\\click\\core.py\", line 1443, in invoke\n    return ctx.invoke(self.callback, **ctx.params)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\workspace\\opensource\\dify_ws\\dify\\api\\.venv\\Lib\\site-packages\\click\\core.py\", line 788, in invoke\n    return __callback(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\workspace\\opensource\\dify_ws\\dify\\api\\.venv\\Lib\\site-packages\\click\\decorators.py\", line 33, in new_func\n    return f(get_current_context(), *args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\workspace\\opensource\\dify_ws\\dify\\api\\.venv\\Lib\\site-packages\\flask\\cli.py\", line 400, in decorator\n    return ctx.invoke(f, *args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\workspace\\opensource\\dify_ws\\dify\\api\\.venv\\Lib\\site-packages\\click\\core.py\", line 788, in invoke\n    return __callback(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\workspace\\opensource\\dify_ws\\dify\\api\\.venv\\Lib\\site-packages\\flask_migrate\\cli.py\", line 154, in upgrade\n    _upgrade(directory, revision, sql, tag, x_arg)\n  File \"D:\\workspace\\opensource\\dify_ws\\dify\\api\\.venv\\Lib\\site-packages\\flask_migrate\\__init__.py\", line 111, in wrapped\n    f(*args, **kwargs)\n  File \"D:\\workspace\\opensource\\dify_ws\\dify\\api\\.venv\\Lib\\site-packages\\flask_migrate\\__init__.py\", line 200, in upgrade\n    command.upgrade(config, revision, sql=sql, tag=tag)\n  File \"D:\\workspace\\opensource\\dify_ws\\dify\\api\\.venv\\Lib\\site-packages\\alembic\\command.py\", line 406, in upgrade\n    script.run_env()\n  File \"D:\\workspace\\opensource\\dify_ws\\dify\\api\\.venv\\Lib\\site-packages\\alembic\\script\\base.py\", line 586, in run_env\n    util.load_python_file(self.dir, \"env.py\")\n  File \"D:\\workspace\\opensource\\dify_ws\\dify\\api\\.venv\\Lib\\site-packages\\alembic\\util\\pyfiles.py\", line 95, in load_python_file\n    module = load_module_py(module_id, path)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\workspace\\opensource\\dify_ws\\dify\\api\\.venv\\Lib\\site-packages\\alembic\\util\\pyfiles.py\", line 113, in load_module_py\n    spec.loader.exec_module(module)  # type: ignore\n    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"<frozen importlib._bootstrap_external>\", line 995, in exec_module\n  File \"<frozen importlib._bootstrap>\", line 488, in _call_with_frames_removed\n  File \"D:\\workspace\\opensource\\dify_ws\\dify\\api\\migrations\\env.py\", line 109, in <module>\n    run_migrations_online()\n  File \"D:\\workspace\\opensource\\dify_ws\\dify\\api\\migrations\\env.py\", line 93, in run_migrations_online\n    with connectable.connect() as connection:\n         ^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\workspace\\opensource\\dify_ws\\dify\\api\\.venv\\Lib\\site-packages\\sqlalchemy\\engine\\base.py\", line 3278, in connect\n    return self._connection_cls(self)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\workspace\\opensource\\dify_ws\\dify\\api\\.venv\\Lib\\site-packages\\sqlalchemy\\engine\\base.py\", line 146, in __init__\n    self._dbapi_connection = engine.raw_connection()\n                             ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\workspace\\opensource\\dify_ws\\dify\\api\\.venv\\Lib\\site-packages\\sqlalchemy\\engine\\base.py\", line 3302, in raw_connection\n    return self.pool.connect()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"D:\\workspace\\opensource\\dify_ws\\dify\\api\\.venv\\Lib\\site-packages\\sqlalchemy\\pool\\base.py\", line 449, in connect\n    return _ConnectionFairy._checkout(self)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\workspace\\opensource\\dify_ws\\dify\\api\\.venv\\Lib\\site-packages\\sqlalchemy\\pool\\base.py\", line 1263, in _checkout\n    fairy = _ConnectionRecord.checkout(pool)\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\workspace\\opensource\\dify_ws\\dify\\api\\.venv\\Lib\\site-packages\\sqlalchemy\\pool\\base.py\", line 712, in checkout\n    rec = pool._do_get()\n          ^^^^^^^^^^^^^^\n  File \"D:\\workspace\\opensource\\dify_ws\\dify\\api\\.venv\\Lib\\site-packages\\sqlalchemy\\pool\\impl.py\", line 179, in _do_get\n    with util.safe_reraise():\n         ^^^^^^^^^^^^^^^^^^^\n  File \"D:\\workspace\\opensource\\dify_ws\\dify\\api\\.venv\\Lib\\site-packages\\sqlalchemy\\util\\langhelpers.py\", line 146, in __exit__\n    raise exc_value.with_traceback(exc_tb)\n  File \"D:\\workspace\\opensource\\dify_ws\\dify\\api\\.venv\\Lib\\site-packages\\sqlalchemy\\pool\\impl.py\", line 177, in _do_get\n    return self._create_connection()\n           ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\workspace\\opensource\\dify_ws\\dify\\api\\.venv\\Lib\\site-packages\\sqlalchemy\\pool\\base.py\", line 390, in _create_connection\n    return _ConnectionRecord(self)\n           ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\workspace\\opensource\\dify_ws\\dify\\api\\.venv\\Lib\\site-packages\\sqlalchemy\\pool\\base.py\", line 674, in __init__\n    self.__connect()\n  File \"D:\\workspace\\opensource\\dify_ws\\dify\\api\\.venv\\Lib\\site-packages\\sqlalchemy\\pool\\base.py\", line 900, in __connect\n    with util.safe_reraise():\n         ^^^^^^^^^^^^^^^^^^^\n  File \"D:\\workspace\\opensource\\dify_ws\\dify\\api\\.venv\\Lib\\site-packages\\sqlalchemy\\util\\langhelpers.py\", line 146, in __exit__\n    raise exc_value.with_traceback(exc_tb)\n  File \"D:\\workspace\\opensource\\dify_ws\\dify\\api\\.venv\\Lib\\site-packages\\sqlalchemy\\pool\\base.py\", line 896, in __connect\n    self.dbapi_connection = connection = pool._invoke_creator(self)\n                                         ^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\workspace\\opensource\\dify_ws\\dify\\api\\.venv\\Lib\\site-packages\\sqlalchemy\\engine\\create.py\", line 643, in connect\n    return dialect.connect(*cargs, **cparams)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\workspace\\opensource\\dify_ws\\dify\\api\\.venv\\Lib\\site-packages\\sqlalchemy\\engine\\default.py\", line 621, in connect\n    return self.loaded_dbapi.connect(*cargs, **cparams)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\workspace\\opensource\\dify_ws\\dify\\api\\.venv\\Lib\\site-packages\\psycopg2\\__init__.py\", line 122, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nUnicodeDecodeError: 'utf-8' codec can't decode byte 0xd6 in position 61: invalid continuation byte\n\n(non-package-mode-py3.12) D:\\workspace\\opensource\\dify_ws\\dify\\api>\n```\n\n### ‚úîÔ∏è Expected Behavior\n\n_No response_\n\n### ‚ùå Actual Behavior\n\n_No response_",
      "closed_by": {
        "login": "crazywoola",
        "id": 100913391,
        "node_id": "U_kgDOBgPQ7w",
        "avatar_url": "https://avatars.githubusercontent.com/u/100913391?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/crazywoola",
        "html_url": "https://github.com/crazywoola",
        "followers_url": "https://api.github.com/users/crazywoola/followers",
        "following_url": "https://api.github.com/users/crazywoola/following{/other_user}",
        "gists_url": "https://api.github.com/users/crazywoola/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/crazywoola/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/crazywoola/subscriptions",
        "organizations_url": "https://api.github.com/users/crazywoola/orgs",
        "repos_url": "https://api.github.com/users/crazywoola/repos",
        "events_url": "https://api.github.com/users/crazywoola/events{/privacy}",
        "received_events_url": "https://api.github.com/users/crazywoola/received_events",
        "type": "User",
        "user_view_type": "public",
        "site_admin": false
      },
      "reactions": {
        "url": "https://api.github.com/repos/langgenius/dify/issues/14199/reactions",
        "total_count": 1,
        "+1": 1,
        "-1": 0,
        "laugh": 0,
        "hooray": 0,
        "confused": 0,
        "heart": 0,
        "rocket": 0,
        "eyes": 0
      },
      "timeline_url": "https://api.github.com/repos/langgenius/dify/issues/14199/timeline",
      "performed_via_github_app": null,
      "state_reason": "completed"
    }
  ],
  "created_at": 1749736669.5149367,
  "expires_at": 1749740269.5149372
}